---
layout: page
title: Quellen
permalink: /quellen/
---

Nachfolgend ist die verwendete Literatur kapitelweise aufgeführt. Klicken Sie den Kapitelnamen, um die Liste aufzublättern.

<details>
  <summary>Einleitung</summary>

* _Rozenblit L_ & _Keil F_ (2002) **The misunderstood limits of folk science: An illusion of explanatory depth**. Cognitive Science 26:521-562, doi: [10.1207/s15516709cog2605_1](https://doi.org/10.1207/s15516709cog2605_1)

* _Kruger J_ & _Dunning D_ (1999) **Unskilled and unaware of it: How difficulties in recognizing one's own incompetence lead to inflated self-assessments.** Journal of Personality and Social Psychology 77:1121-1134, doi: [10.1037/0022-3514.77.6.1121](https://doi.org/10.1037/0022-3514.77.6.1121)

* _Nishimasu H_, _Ran FA_ _et&nbsp;al._ (2014) **Crystal Structure of Cas9 in Complex with Guide RNA and Target DNA**. Cell 156:935-949, doi: [10.1016/j.cell.2014.02.001](https://doi.org/10.1016/j.cell.2014.02.001)

* _Wünschiers R_ (2019) **Generation Gen-Schere: Wie begegnen wir der gentechnologischen Revolution**. Springer, Berlin, ISBN 978-3662590478, doi: [10.1038/s41562-018-0520-3](https://doi.org/10.1038/s41562-018-0520-3)

* _Robinson T_ (2017) **Genetik für Dummies**. Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim, ISBN 978-3-527-71417-9

* _Fernbach PM_, _Light N_ _et&nbsp;al._ (2019) **Extreme opponents of genetically modified foods know the least but think they know the most**. Nat Hum Behav 3:251-256, doi: [10.1038/s41562-018-0520-3](https://doi.org/10.1038/s41562-018-0520-3)

* _Küntzel K_ (2019) **Nachhaltig & ökologisch Leben für Dummies**. John Wiley & Sons, Incorporated, Newark, ISBN 978-3-527-82178-5

* _NASA_, **Images on the golden record**. Zugegriffen unter [Internetlink](https://voyager.jpl.nasa.gov/golden-record/whats-on-the-record/images/), zugegriffen am 19/01/21
</details>


<details>
  <summary>Kapitel 1: Die Welt der Gene</summary>

* _Chargaff E_ (2003) **Stimmen im Labyrinth: Drei Dialoge über die Natur und ihre Erforschung**. Klett-Cotta, Stuttgart, ISBN 978-3608935806

* _Watson JD_ & _Crick FHC_ (1953) **Molecular Structure of Nucleic Acids: A Structure for Deoxyribose Nucleic Acid**. Nature 171:737-738, doi: [10.1038/171737a0](https://doi.org/10.1038/171737a0)

* _Redaktion_ (2007), **Labor suspendiert Nobelpreisträger Watson**. Zugegriffen unter [Internetlink](https://www.spiegel.de/wissenschaft/mensch/rassistische-aeusserungen-labor-suspendiert-nobelpreistraeger-watson-a-512455.html), zugegriffen am 30/11/20

* &#8212; (2014), **Goldmedaille für 4,7 Millionen Dollar versilbert**. Zugegriffen unter [Internetlink](https://www.spiegel.de/panorama/leute/nobelpreis-medaille-von-james-watson-fuer-4-7-millionen-dollar-versteigert-a-1006728.html), zugegriffen am 30/11/20

* &#8212; (2019), **Nobelpreisträger Watson verliert Ehrentitel wegen rassistischer Äußerungen**. Zugegriffen unter [Internetlink](https://www.spiegel.de/wissenschaft/medizin/james-watson-nobelpreistraeger-verliert-titel-wegen-rassismus-a-1247977.html), zugegriffen am 30/11/20

* _Sender R_, _Fuchs S_ & _Milo R_ (2016) **Revised Estimates for the Number of Human and Bacteria Cells in the Body. PLOS** Biology 14:e1002533, doi: [10.1371/journal.pbio.1002533](https://doi.org/10.1371/journal.pbio.1002533)

* _Sender R_ & _Milo R_ (2021) **The distribution of cellular turnover in the human body**. Nat Med 27:45-48, doi: [10.1038/s41591-020-01182-9](https://doi.org/10.1038/s41591-020-01182-9)

* _Preston BD_, _Albertson TM_ & _Herr AJ_ (2010) **DNA replication fidelity and cancer**. Seminars in Cancer Biology 20:281-293, doi: [10.1016/j.semcancer.2010.10.009](https://doi.org/10.1016/j.semcancer.2010.10.009)

* _de Vries H_ (1901) **Die Mutationstheorie. Versuche und Beobachtungen über die Entstehung von Arten im Pflanzenreich**. Veit & Comp., doi: [10.5962/bhl.title.11336](https://doi.org/10.5962/bhl.title.11336)

* _McClintock B_ (1931) **The Order of the Genes C, Sh and Wx in Zea Mays with Reference to a Cytologically Known Point in the Chromosome**. Proc Natl Acad Sci USA 17:485-491, doi: [10.1073/pnas.17.8.485](https://doi.org/10.1073/pnas.17.8.485)

* _Creighton HB_ & _McClintock B_ (1931) **A Correlation of Cytological and Genetical Crossing-Over in Zea Mays**. Proc Natl Acad Sci USA 17:492-497, doi: [10.1073/pnas.17.8.492](https://doi.org/10.1073/pnas.17.8.492)

* _Wolpert L_, _Ghysen A_ & _García-Bellido A_ (1998) **Debatable issues**. The International journal of developmental biology 42:511-518

* _Mendel G_ (1866) **Versuche über Pflanzen-Hybriden.** Verhandlungen des Naturforschenden Vereins zu Brünn 4:3-47, doi: [10.5962/bhl.title.61004](https://doi.org/10.5962/bhl.title.61004)

* _Bateson W_ (1902) **Mendel's principles of heredity: A defence.** University Press, doi: [10.5962/bhl.title.46238](https://doi.org/10.5962/bhl.title.46238)

* _Brem G_ (Editor) (2017) **150 Jahre Mendelsche Regeln: Vom Erbsenzählen zum Gen-Editieren**, Nova Acta Leopoldina, Deutsche Akademie der Naturforscher Leopoldina - Nationale Akademie der Wissenschaften: Wissenschaftliche Verlagsgesellschaft, ISBN 978-3-8047-3703-7

* _Johannsen W_ (1909) **Elemente der exakten Erblichkeitslehre**. Verlag von Gustav Fischer, Jena, doi: [10.5962/bhl.title.1060](https://doi.org/10.5962/bhl.title.1060)

* _Stotz K_ & _Griffiths P_ (2004) **Genes: Philosophical Analyses Put to the Test**. History and Philosophy of the Life Sciences 26:5-28, doi: [10.2307/23333378](https://doi.org/10.2307/23333378)

* _Pearson H_ (2006) **What is a gene?** Nature 441:398-401, doi: [10.1038/441398a](https://doi.org/10.1038/441398a)

* _Delmore KE_, _Toews DP_ _et&nbsp;al._ (2016) **The Genetics of Seasonal Migration and Plumage Color**. Current Biology 26:2167-2173, doi: [10.1016/j.cub.2016.06.015](https://doi.org/10.1016/j.cub.2016.06.015)

* _Dolgin E_ (2017) **The most popular genes in the human genome**. Nature 551:427-431, doi: [10.1038/d41586-017-07291-9](https://doi.org/10.1038/d41586-017-07291-9)

* _Armstrong S_ (2014) **P53: The gene that cracked the cancer code**. Bloomsbury Sigma, London, ISBN 978-1-4729-1053-0

* _Knudson AG_ (1971) **Mutation and Cancer: Statistical Study of Retinoblastoma**. Proc Natl Acad Sci USA 68:820-823, doi: [10.1073/pnas.68.4.820](https://doi.org/10.1073/pnas.68.4.820)

* _Finlay CA_, _Hinds PW_ & _Levine AJ_ (1989) **The p53 proto-oncogene can act as a suppressor of transformation**. Cell 57:1083-1093, doi: [10.1016/0092-8674(89)90045-7](https://doi.org/10.1016/0092-8674(89)90045-7)

* _Belyi VA_, _Ak P_ _et&nbsp;al._ (2009) **The Origins and Evolution of the p53 Family of Genes**. Cold Spring Harbor Perspectives in Biology 2:a001198-a001198, doi: [10.1101/cshperspect.a001198](https://doi.org/10.1101/cshperspect.a001198)

* _Wagener C_ & _Müller O_ (2009) **Molekulare Onkologie**. Thieme, Stuttgart, 3rd ed., ISBN 9783131035134

* _Kim EB_, _Fang X_ _et&nbsp;al._ (2011) **Genome sequencing reveals insights into physiology and longevity of the naked mole rat**. Nature 479:223-227, doi: [10.1038/nature10533](https://doi.org/10.1038/nature10533)

* _Deuker MM_, _Lewis KN_ _et&nbsp;al._ (2020) **Unprovoked Stabilization and Nuclear Accumulation of the Naked Mole-Rat p53 Protein**. Scientific Reports 10, doi: [10.1038/s41598-020-64009-0](https://doi.org/10.1038/s41598-020-64009-0)

* _Goethe J_ (2016) **Gedichte: Ausgabe letzter Hand 1827**. Hofenberg, ISBN 978-3-8430-1760-2

* _Pozhitkov AE_, _Neme R_ _et&nbsp;al._ (2017) **Tracing the dynamics of gene transcripts after organismal death**. Open Biology 7:160267, doi: [10.1098/rsob.160267](https://doi.org/10.1098/rsob.160267)

* _Ussery DW_ & _Structure D_ (2002), **A-, B-and Z-DNA Helix Families, Encyclopedia of Life Sciences**. doi: [10.1038/npg.els.0003122](https://doi.org/10.1038/npg.els.0003122)

* _Rutherford A_, **Dna's twist to the right is not to be meddled with, so let's lose the lefties**. Zugegriffen unter [Internetlink](https://www.theguardian.com/science/blog/2013/apr/30/dna-twist-to-right), zugegriffen am 30/11/20

* _Schneider T_, **The left handed dna hall of fame**. Zugegriffen unter [Internetlink](http://www.fred.net/tds/leftdna/), zugegriffen am 30/11/20

* _Woese CR_ & _Fox GE_ (1977) **Phylogenetic structure of the prokaryotic domain: The primary kingdoms**. Proc Natl Acad Sci USA 74:5088-5090, doi: [10.1073/pnas.74.11.5088](https://doi.org/10.1073/pnas.74.11.5088)

* _Andersen KG_, _Rambaut A_ _et&nbsp;al._ (2020) **The proximal origin of SARS-CoV-2**. Nature Medicine 26:450-452, doi: [10.1038/s41591-020-0820-9](https://doi.org/10.1038/s41591-020-0820-9)

* _Hou W_ (2020) **Characterization of codon usage pattern in SARS-CoV-2**. Virol J 17, doi: [10.1186/s12985-020-01395-x](https://doi.org/10.1186/s12985-020-01395-x)

* _The International SNP Map Working Group_ (2001) **A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms**. Nature 409:928-933, doi: [10.1038/35057149](https://doi.org/10.1038/35057149)

* _Ingram VM_ (1956) **A Specific Chemical Difference Between the Globins of Normal Human and Sickle-Cell Anämia Hämoglobin**. Nature 178:792-794, doi: [10.1038/178792a0](https://doi.org/10.1038/178792a0)

* _Chang JC_ & _Kan YW_ (1979) **Beta zero thalassemia, a nonsense mutation in man.** Proc Natl Acad Sci USA 76:2886-2889, doi: [10.1073/pnas.76.6.2886](https://doi.org/10.1073/pnas.76.6.2886)

* _Hamosh A_, _King T_ _et&nbsp;al._ (1992) **Cystic fibrosis patients bearing both the common missense mutation Gly&gt;Asp at codon 551 and the delta F508 mutation are clinically indistinguishable from delta F508 homozygotes, except for decreased risk of meconium ileus.** American journal of human genetics 51 2:245-50

* _Wolf AB_, _Caselli RJ_ _et&nbsp;al._ (2013) **APOE and neuroenergetics: An emerging paradigm in Alzheimers disease**. Neurobiology of Aging 34:1007-1017, doi: [10.1016/j.neurobiolaging.2012.10.011](https://doi.org/10.1016/j.neurobiolaging.2012.10.011)

* _Vilarrasa-Blasi R_, _Soler-Vila P_ _et&nbsp;al._ (2021) **Dynamics of genome architecture and chromatin function during human B cell differentiation and neoplastic transformation**. Nat Commun 12, doi: [10.1038/s41467-020-20849-y](https://doi.org/10.1038/s41467-020-20849-y)
</details>


<details>
  <summary>Kapitel 2: Von der Vererbung zur quantitativen Genetik</summary>

* _Szabó A_ (2017) **Ursprung des Begriffs Genetik und seine Verwendung vor und nach Mendel**. Nova Acta Leopoldina 65-79

* _Ellis TN_, _Hofer JM_ _et&nbsp;al._ (2011) **Mendel, 150 years on**. Trends in Plant Science 16:590-596, doi: [10.1016/j.tplants.2011.06.006](https://doi.org/10.1016/j.tplants.2011.06.006)

* _Jollos V_ (1921) **Experimentelle Protistenstudien**. G. Fischer,, doi: [10.5962/bhl.title.11942](https://doi.org/10.5962/bhl.title.11942)

* _Kahneman D_ (2012) **Schnelles Denken, langsames Denken**. Siedler, München, ISBN 9783886808861

* _Galton F_ (1886) **Regression Towards Mediocrity in Hereditary Stature.** The Journal of the Anthropological Institute of Great Britain and Ireland 15:246, doi: [10.2307/2841583](https://doi.org/10.2307/2841583)

* _Wikipedia_, **Regression toward the mean**. Zugegriffen unter [Internetlink](https://en.wikipedia.org/wiki/Regression_toward_the_mean), zugegriffen am 22/03/21

* _Boyle EA_, _Li YI_ & _Pritchard JK_ (2017) **An Expanded View of Complex Traits: From Polygenic to Omnigenic**. Cell 169:1177-1186, doi: [10.1016/j.cell.2017.05.038](https://doi.org/10.1016/j.cell.2017.05.038)

* _Mähler N_, _Schiffthaler B_ _et&nbsp;al._ (2020) **Leaf shape in Populus tremula is a complex, omnigenic trait**. Ecology and Evolution 10:11922-11940, doi: [10.1002/ece3.6691](https://doi.org/10.1002/ece3.6691)

* _Galton F_ (1883) **Inquiries into human faculty and its development.** MacMillan Co, doi: [10.1037/14178-000](https://doi.org/10.1037/14178-000)

* _Ploetz A_ (1895) **Grundlinien einer Rassen-Hygiene: Die Tüchtigkeit unsrer Rasse und der Schutz der Schwachen: Ein Versuch über Rassenhygiene und ihr Verhältniss zu den humanen Idealen, besonders zum Socialismus**. Fischer
</details>

<details>
  <summary>Kapitel 3: Vom Gen über’s Genom zum Ich</summary>

* _Slonim N_, _Bilu Y_ _et&nbsp;al._ (2021) **An autonomous debating system**. Nature 591:379-384, doi: [10.1038/s41586-021-03215-w](https://doi.org/10.1038/s41586-021-03215-w)

* _Rebout N_, _Lone JC_ _et&nbsp;al._ (2021) **Measuring complexity in organisms and organizations**. R Soc open sci 8, doi: [10.1098/rsos.200895](https://doi.org/10.1098/rsos.200895)

* _Waring M_ & _Britten RJ_ (1966) **Nucleotide Sequence Repetition: A Rapidly Reassociating Fraction of Mouse DNA**. Science 154:791-794, doi: [10.1126/science.154.3750.791](https://doi.org/10.1126/science.154.3750.791)

* _Peterson DG_, _Wessler SR_ & _Paterson AH_ (2002) **Efficient capture of unique sequences from eukaryotic genomes**. Trends in Genetics 18:547-550, doi: [10.1016/s0168-9525(02)02764-6](https://doi.org/10.1016/s0168-9525(02)02764-6)

* _Van Valen L_ (1973) **A new evolutionary law**. Evolutionary theory 1:1-30

* _Carroll L_ (1998) **Alice hinter den Spiegeln**. Insel-Verlag, Frankfurt am Main, ISBN 9783458317975

* _Weber-Lehmann J_, _Schilling E_ _et&nbsp;al._ (2014) **Finding the needle in the haystack: Differentiating "identical" twins in paternity testing and forensics by ultra-deep next generation sequencing**. Forensic Science International: Genetics 9:42-46, doi: [10.1016/j.fsigen.2013.10.015](https://doi.org/10.1016/j.fsigen.2013.10.015)

* _Fontdevila A_ (2011) **The dynamic genome: A Darwinian approach**. Oxford University Press, New York, ISBN 978-0199541379

* _Carretero-Paulet L_, _Librado P_ _et&nbsp;al._ (2015) **High Gene Family Turnover Rates and Gene Space Adaptation in the Compact Genome of the Carnivorous Plant Utricularia gibba**. Molecular Biology and Evolution 32:1284-1295, doi: [10.1093/molbev/msv020](https://doi.org/10.1093/molbev/msv020)

* _Bodea GO_, _McKelvey EGZ_ & _Faulkner GJ_ (2018) **Retrotransposon-induced mosaicism in the neural genome**. Open Biol 8:180074, doi: [10.1098/rsob.180074](https://doi.org/10.1098/rsob.180074)

* _Locke DP_, _Hillier LW_ _et&nbsp;al._ (2011) **Comparative and demographic analysis of orang-utan genomes**. Nature 469:529-533, doi: [10.1038/nature09687](https://doi.org/10.1038/nature09687)

* _Payer LM_, _Steranka JP_ _et&nbsp;al._ (2017) **Structural variants caused by Alu insertions are associated with risks for many human diseases**. Proc Natl Acad Sci USA 114:E3984-E3992, doi: [10.1073/pnas.1704117114](https://doi.org/10.1073/pnas.1704117114)

* _Comas D_, _Plaza S_ _et&nbsp;al._ (2001) **Recent Insertion of an Alu Element Within a Polymorphic Human-Specific Alu Insertion**. Molecular Biology and Evolution 18:85-88, doi: [10.1093/oxfordjournals.molbev.a003722](https://doi.org/10.1093/oxfordjournals.molbev.a003722)

* _Cordaux R_ & _Batzer MA_ (2009) **The impact of retrotransposons on human genome evolution**. Nat Rev Genet 10:691-703, doi: [10.1038/nrg2640](https://doi.org/10.1038/nrg2640)

* _Touchon M_, _Hoede C_ _et&nbsp;al._ (2009) **Organised Genome Dynamics in the Escherichia coli Species Results in Highly Diverse Adaptive Paths**. PLoS Genetics 5:e1000344, doi: [10.1371/journal.pgen.1000344](https://doi.org/10.1371/journal.pgen.1000344)

* _Stokes HW_ & _Gillings MR_ (2011) **Gene flow, mobile genetic elements and the recruitment of antibiotic resistance genes into Gram-negative pathogens**. FEMS Microbiology Reviews 35:790-819, doi: [10.1111/j.1574-6976.2011.00273.x](https://doi.org/10.1111/j.1574-6976.2011.00273.x)

* _Stegemann S_ & _Bock R_ (2009) **Exchange of Genetic Material Between Cells in Plant Tissue Grafts**. Science 324:649-651, doi: [10.1126/science.1170397](https://doi.org/10.1126/science.1170397)

* _Sjölund M_, _Bonnedahl J_ _et&nbsp;al._ (2008) **Dissemination of Multidrug-Resistant Bacteria into the Arctic**. Emerg Infect Dis 14:70-72, doi: [10.3201/eid1401.070704](https://doi.org/10.3201/eid1401.070704)

* _Bartoloni A_, _Pallecchi L_ _et&nbsp;al._ (2009) **Antibiotic resistance in a very remote Amazonas community**. International Journal of Antimicrobial Agents 33:125-129, doi: [10.1016/j.ijantimicag.2008.07.029](https://doi.org/10.1016/j.ijantimicag.2008.07.029)

* _Clemente JC_, _Pehrsson EC_ _et&nbsp;al._ (2015) **The microbiome of uncontacted Amerindians**. Sci Adv 1:e1500183, doi: [10.1126/sciadv.1500183](https://doi.org/10.1126/sciadv.1500183)

* _Dunning LT_, _Olofsson JK_ _et&nbsp;al._ (2019) **Lateral transfers of large DNA fragments spread functional genes among grasses**. Proc Natl Acad Sci USA 116:4416-4425, doi: [10.1073/pnas.1810031116](https://doi.org/10.1073/pnas.1810031116)

* _H_uman Genome Sequencing Consortium I (2001) **Initial sequencing and analysis of the human genome**. Nature 409:860-921, doi: [10.1038/35057062](https://doi.org/10.1038/35057062)

* _Venter JC_, _Adams MD_ _et&nbsp;al._ (2001) **The Sequence of the Human Genome**. Science 291:1304-1351, doi: [10.1126/science.1058040](https://doi.org/10.1126/science.1058040)

* _Miga KH_, _Koren S_ _et&nbsp;al._ (2020) **Telomere-to-telomere assembly of a complete human X chromosome**. Nature 585:79-84, doi: [10.1038/s41586-020-2547-7](https://doi.org/10.1038/s41586-020-2547-7)

* _Reardon S_ (2021) **A complete human genome sequence is close: How scientists filled in the gaps**. Nature 594:158-159, doi: [10.1038/d41586-021-01506-w](https://doi.org/10.1038/d41586-021-01506-w)

* _Karki R_, _Pandya D_ _et&nbsp;al._ (2015) **Defining "mutation" and "polymorphism" in the era of personal genomics**. BMC Med Genomics Medical Genomics 8, doi: [10.1186/s12920-015-0115-z](https://doi.org/10.1186/s12920-015-0115-z)

* _Ebert P_, _Audano PA_ _et&nbsp;al._ (2021) **Haplotype-resolved diverse human genomes and integrated analysis of structural variation**. Science eabf7117, doi: [10.1126/science.abf7117](https://doi.org/10.1126/science.abf7117)

* _Sudmant PH_,  _et&nbsp;al._ (2015) **An integrated map of structural variation in 2,504 human genomes**. Nature 526:75-81, doi: [10.1038/nature15394](https://doi.org/10.1038/nature15394)

* _Abel HJ_, _Larson DE_ _et&nbsp;al._ (2018) **Mapping and characterization of structural variation in 17,795 deeply sequenced human genomes**. BioRxiv doi: [10.1101/508515](https://doi.org/10.1101/508515)

* _Lin Z_ (2004) **Genetics: Genomic Research and Human Subject Privacy**. Science 305:183-183, doi: [10.1126/science.1095019](https://doi.org/10.1126/science.1095019)

* _Consortium TGP_ (2015) **A global reference for human genetic variation**. Nature 526:68-74, doi: [10.1038/nature15393](https://doi.org/10.1038/nature15393)

* &#8212; (2010) **A map of human genome variation from population-scale sequencing**. Nature 467:1061-1073, doi: [10.1038/nature09534](https://doi.org/10.1038/nature09534)

* _Ng SB_, _Turner EH_ _et&nbsp;al._ (2009) **Targeted capture and massively parallel sequencing of 12 human exomes**. Nature 461:272-276, doi: [10.1038/nature08250](https://doi.org/10.1038/nature08250)

* _Borek E_ (1973) **The sculpture of life**. Columbia University Press, London, ISBN 9780231083348

* _Cooper A_ & _Fortey R_ (1998) **Evolutionary explosions and the phylogenetic fuse**. Trends in Ecology and Evolution Evolution 13:151-156, doi: [10.1016/s0169-5347(97)01277-9](https://doi.org/10.1016/s0169-5347(97)01277-9)

* _Hueber SD_, _Weiller GF_ _et&nbsp;al._ (2010) **Improving Hox Protein Classification across the Major Model Organisms. PLoS ONE ONE** 5:e10820, doi: [10.1371/journal.pone.0010820](https://doi.org/10.1371/journal.pone.0010820)

* _Haughn GW_ & _Somerville CR_ (1988) **Genetic control of morphogenesis inArabidopsis**. Dev Genet 9:73-89, doi: [10.1002/dvg.1020090202](https://doi.org/10.1002/dvg.1020090202)

* _Turing A_ (1952) **The chemical basis of morphogenesis**. Phil Trans R Soc Lond B 237:37-72, doi: [10.1098/rstb.1952.0012](https://doi.org/10.1098/rstb.1952.0012)

* _Tung A_ & _Levin M_ (2020) **Extra-genomic instructive influences in morphogenesis: A review of external signals that regulate growth and form**. Developmental Biology 461:1-12, doi: [10.1016/j.ydbio.2020.01.010](https://doi.org/10.1016/j.ydbio.2020.01.010)

* _Pinet K_ & _McLaughlin KA_ (2019) **Mechanisms of physiological tissue remodeling in animals: Manipulating tissue, organ, and organism morphology**. Developmental Biology 451:134-145, doi: [10.1016/j.ydbio.2019.04.001](https://doi.org/10.1016/j.ydbio.2019.04.001)

* _Maden M_ (2008) **Axolotl/Newt.** In: Methods In Molecular Biology, 467-480, Humana Press, doi: [10.1007/978-1-60327-483-8_32](https://doi.org/10.1007/978-1-60327-483-8_32)

* _Brown FA_ & _Chow CS_ (1973) **Interorganismic and Environmental Influences Through Extremely Weak Electromagnetic Fields**. The Biological Bulletin 144:437-461, doi: [10.2307/1540299](https://doi.org/10.2307/1540299)

* _Harrison CGA_ & _Funnell BM_ (1964) **Relationship of Palaeomagnetic Reversals and Micropalaeontology in Two Late Caenozoic Cores from the Pacific Ocean**. Nature 204:566-566, doi: [10.1038/204566a0](https://doi.org/10.1038/204566a0)
</details>

<details>
  <summary>Kapitel 4: Safety First: Gentechnik und Gesetze</summary>

* _Berg P_, _Baltimore D_ _et&nbsp;al._ (1974) **Potential Biohazards of Recombinant DNA Molecules**. Science 185:303-303, doi: [10.1126/science.185.4148.303](https://doi.org/10.1126/science.185.4148.303)

* &#8212; (1975) **Summary statement of the Asilomar conference on recombinant DNA molecules.** Proc Natl Acad Sci USA 72:1981-1984, doi: [10.1073/pnas.72.6.1981](https://doi.org/10.1073/pnas.72.6.1981)
</details>

<details>
  <summary>Kapitel 5: Drei Gene verändern die Welt</summary>

* _Suttle CA_ (2005) **Viruses in the sea**. Nature 437:356-361, doi: [10.1038/nature04160](https://doi.org/10.1038/nature04160)

* _Camarillo-Guerrero LF_, _Almeida A_ _et&nbsp;al._ (2021) **Massive expansion of human gut bacteriophage diversity**. Cell 184:1098-1109.e9, doi: [10.1016/j.cell.2021.01.029](https://doi.org/10.1016/j.cell.2021.01.029)

* _Kessler C_ & _Manta V_ (1990) **Specificity of restriction endonucleases and DNA modification methyltransferases – a review (edition 3)**. Gene 92:1-240, doi: [10.1016/0378-1119(90)90486-b](https://doi.org/10.1016/0378-1119(90)90486-b)

* _Arber W_ & _Linn S_ (1969) **DNA Modification and Restriction**. Annual Review of Biochemistry 38:467-500, doi: [10.1146/annurev.bi.38.070169.002343](https://doi.org/10.1146/annurev.bi.38.070169.002343)

* _Lederberg J_ (1952) **Cell Genetics and Hereditary Symbiosis**. Physiological Reviews 32:403-430, doi: [10.1152/physrev.1952.32.4.403](https://doi.org/10.1152/physrev.1952.32.4.403)

* _Chang ACY_ & _Cohen SN_ (1974) **Genome Construction Between Bacterial Species In Vitro: Replication and Expression of Staphylococcus Plasmid Genes in Escherichia coli**. Proc Natl Acad Sci USA 71:1030-1034, doi: [10.1073/pnas.71.4.1030](https://doi.org/10.1073/pnas.71.4.1030)

* _Pappalardo AM_, _Petraccioli A_ _et&nbsp;al._ (2019) **From Fish Eggs to Fish Name: Caviar Species Discrimination by COIBar-RFLP, an Efficient Molecular Approach to Detect Fraud in the Caviar Trade**. Molecules 24:2468, doi: [10.3390/molecules24132468](https://doi.org/10.3390/molecules24132468)

* _Ropka-Molik K_, _Stefaniuk-Szmukier M_ _et&nbsp;al._ (2019) **The use of the SLC16A1 gene as a potential marker to predict race performance in Arabian horses. BMC** Genetics 20, doi: [10.1186/s12863-019-0774-4](https://doi.org/10.1186/s12863-019-0774-4)

* _Lehman IR_, _Bessman MJ_ _et&nbsp;al._ (1958) **Deoxyribonucleic acid polymerase from the extreme thermophile Thermus aquaticus.** Biol Chem 233:163-170

* _Klenow H_ & _Henningsen I_ (1970) **Selective Elimination of the Exonuclease Activity of the Deoxyribonucleic Acid Polymerase from Escherichia coli B by Limited Proteolysis**. Proc Natl Acad Sci USA 65:168-175, doi: [10.1073/pnas.65.1.168](https://doi.org/10.1073/pnas.65.1.168)

* _Klenow H_ (2003) **Some Selected Recollections from a Life with Biochemistry.** In: Comprehensive Biochemistry, 285-317, Elsevier, doi: [10.1016/s0069-8032(03)42010-x](https://doi.org/10.1016/s0069-8032(03)42010-x)

* _Chien A_, _Edgar DB_ & _Trela JM_ (1976) **Deoxyribonucleic acid polymerase from the extreme thermophile Thermus aquaticus.** Journal of Bacteriology 127:1550-1557, doi: [10.1128/jb.127.3.1550-1557.1976](https://doi.org/10.1128/jb.127.3.1550-1557.1976)

* _Wheeler DA_, _Srinivasan M_ _et&nbsp;al._ (2008) **The complete genome of an individual by massively parallel DNA sequencing**. Nature 452:872-876, doi: [10.1038/nature06884](https://doi.org/10.1038/nature06884)

* _Levy S_, _Sutton G_ _et&nbsp;al._ (2007) **The Diploid Genome Sequence of an Individual Human**. PLoS Biol Biology 5:e254, doi: [10.1371/journal.pbio.0050254](https://doi.org/10.1371/journal.pbio.0050254)

* _Venter J_ (2009) **Entschlüsselt mein Genom, mein Leben**. S. Fischer, Frankfurt, M, ISBN 3100870301

* _van der Valk T_, _Pečnerová, P_ _et&nbsp;al._ (2021) **Million-year-old DNA sheds light on the genomic history of mammoths**. Nature 591:265-269, doi: [10.1038/s41586-021-03224-9](https://doi.org/10.1038/s41586-021-03224-9)

* _Pečnerová P_, _del Molino DD_ _et&nbsp;al._ (2021) **Mammuthus sp. (Early and Middle Pleistocene Mammoths)**. Trends in Genetics doi: [10.1016/j.tig.2021.04.006](https://doi.org/10.1016/j.tig.2021.04.006)

* _Quick J_, _Loman NJ_ _et&nbsp;al._ (2016) **Real-time, portable genome sequencing for Ebola surveillance**. Nature 530:228-232, doi: [10.1038/nature16996](https://doi.org/10.1038/nature16996)

* _Kafetzopoulou LE_, _Pullan ST_ _et&nbsp;al._ (2019) **Metagenomic sequencing at the epicenter of the Nigeria 2018 Lassa fever outbreak**. Science 363:74-77, doi: [10.1126/science.aau9343](https://doi.org/10.1126/science.aau9343)

* _Walker A_, _Houwaart T_ _et&nbsp;al._ (2020) **Genetic structure of SARS-CoV-2 reflects clonal superspreading and multiple independent introduction events, North-Rhine Westphalia, Germany, February and March 2020**. Eurosurveillance 25, doi: [10.2807/1560-7917.es.2020.25.22.2000746](https://doi.org/10.2807/1560-7917.es.2020.25.22.2000746)

* _Saiki R_, _Scharf S_ _et&nbsp;al._ (1985) **Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia**. Science 230:1350-1354, doi: [10.1126/science.2999980](https://doi.org/10.1126/science.2999980)

* _Mullis KB_ & _Faloona FA_ (1987) **Specific synthesis of DNA in vitro via a polymerase-catalyzed chain reaction.** In: Methods in Enzymology, 335-350, Elsevier, doi: [10.1016/0076-6879(87)55023-6](https://doi.org/10.1016/0076-6879(87)55023-6)

* _de Souza N_ (2011) **Primer: Genome editing with engineered nucleases**. Nat Methods 9:27-27, doi: [10.1038/nmeth.1848](https://doi.org/10.1038/nmeth.1848)

* _Kim YG_, _Cha J_ & _Chandrasegaran S_ (1996) **Hybrid restriction enzymes: Zinc finger fusions to Fok I cleavage domain.** Proc Natl Acad Sci USA 93:1156-1160, doi: [10.1073/pnas.93.3.1156](https://doi.org/10.1073/pnas.93.3.1156)

* _Carroll D_ (2011) **Genome Engineering With Zinc-Finger Nucleases**. Genetics 188:773-782, doi: [10.1534/genetics.111.131433](https://doi.org/10.1534/genetics.111.131433)

* _Hanas JS_, _Hazuda DJ_ _et&nbsp;al._ (1983) **Xenopus transcription factor A requires zinc for binding to the 5 S RNA gene.** Journal of Biological Chemistry 258:14120-14125, doi: [10.1016/s0021-9258(17)43831-2](https://doi.org/10.1016/s0021-9258(17)43831-2)

* _Boch J_ (2011) **TALEs of genome targeting**. Nat Biotechnol 29:135-136, doi: [10.1038/nbt.1767](https://doi.org/10.1038/nbt.1767)

* _Karginov FV_ & _Hannon GJ_ (2010) **The CRISPR System: Small RNA-Guided Defense in Bacteria and Archaea**. Molecular Cell 37:7-19, doi: [10.1016/j.molcel.2009.12.033](https://doi.org/10.1016/j.molcel.2009.12.033)

* _Ishino Y_, _Shinagawa H_ _et&nbsp;al._ (1987) **Nucleotide sequence of the iap gene, responsible for alkaline phosphatase isozyme conversion in Escherichia coli, and identification of the gene product**. J Bacteriol 169:5429-5433, doi: [10.1128/jb.169.12.5429-5433.1987](https://doi.org/10.1128/jb.169.12.5429-5433.1987)

* _Mojica FJM_, _Juez G_ & _Rodriguez-Valera F_ (1993) **Transcription at different salinities of Haloferax mediterranei sequences adjacent to partially modified PstI sites**. Mol Microbiol 9:613-621, doi: [10.1111/j.1365-2958.1993.tb01721.x](https://doi.org/10.1111/j.1365-2958.1993.tb01721.x)

* _Mojica F_, _Ferrer C_ _et&nbsp;al._ (1995) **Long stretches of short tandem repeats are present in the largest replicons of the Archaea Haloferax mediterranei and Haloferax volcanii and could be involved in replicon partitioning**. Mol Microbiol 17:85-93, doi: [10.1111/j.1365-2958.1995.mmi_17010085.x](https://doi.org/10.1111/j.1365-2958.1995.mmi_17010085.x)

* _Mojica FJ_, _Díez-Villaseñor C_ _et&nbsp;al._ (2005) **Intervening Sequences of Regularly Spaced Prokaryotic Repeats Derive from Foreign Genetic Elements**. J Mol Evol 60:174-182, doi: [10.1007/s00239-004-0046-3](https://doi.org/10.1007/s00239-004-0046-3)

* _Jansen R_, _van Embden JDA_ _et&nbsp;al._ (2002) **Identification of genes that are associated with DNA repeats in prokaryotes**. Mol Microbiol 43:1565-1575, doi: [10.1046/j.1365-2958.2002.02839.x](https://doi.org/10.1046/j.1365-2958.2002.02839.x)

* _Makarova KS_, _Grishin NV_ _et&nbsp;al._ (2006) **A putative RNA-interference-based immune system in prokaryotes: Computational analysis of the predicted enzymatic machinery, functional analogies with eukaryotic RNAi, and hypothetical mechanisms of action**. Biol Direct 1:7, doi: [10.1186/1745-6150-1-7](https://doi.org/10.1186/1745-6150-1-7)

* _García-Martínez J_, _Maldonado RD_ _et&nbsp;al._ (2018) **The CRISPR conundrum: Evolve and maybe die, or survive and risk stagnation**. Microb Cell 5:262-268, doi: [10.15698/mic2018.06.634](https://doi.org/10.15698/mic2018.06.634)

* _Barrangou R_, _Fremaux C_ _et&nbsp;al._ (2007) **CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes**. Science 315:1709-1712, doi: [10.1126/science.1138140](https://doi.org/10.1126/science.1138140)

* _Marraffini LA_ (2015) **CRISPR-Cas immunity in prokaryotes**. Nature 526:55-61, doi: [10.1038/nature15386](https://doi.org/10.1038/nature15386)

* _Pawluk A_, _Davidson AR_ & _Maxwell KL_ (2017) **Anti-CRISPR: Discovery, mechanism and function**. Nat Rev Microbiol 16:12-17, doi: [10.1038/nrmicro.2017.120](https://doi.org/10.1038/nrmicro.2017.120)

* _Wagner DL_, _Amini L_ _et&nbsp;al._ (2018) **High prevalence of Streptococcus pyogenes Cas9-reactive T cells within the adult human population**. Nat Med 25:242-248, doi: [10.1038/s41591-018-0204-6](https://doi.org/10.1038/s41591-018-0204-6)

* _Ledford H_ (2020) **CRISPR gene editing in human embryos wreaks chromosomal mayhem**. Nature 583:17-18, doi: [10.1038/d41586-020-01906-4](https://doi.org/10.1038/d41586-020-01906-4)

* _Zuccaro MV_, _Xu J_ _et&nbsp;al._ (2020) **Allele-Specific Chromosome Removal after Cas9 Cleavage in Human Embryos**. Cell 183:1650-1664.e15, doi: [10.1016/j.cell.2020.10.025](https://doi.org/10.1016/j.cell.2020.10.025)

* _Porto EM_, _Komor AC_ _et&nbsp;al._ (2020) **Base editing: Advances and therapeutic opportunities**. Nature Reviews Drug Discovery 19:839-859, doi: [10.1038/s41573-020-0084-6](https://doi.org/10.1038/s41573-020-0084-6)

* _Nuñez JK_, _Chen J_ _et&nbsp;al._ (2021) **Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing**. Cell doi: [10.1016/j.cell.2021.03.025](https://doi.org/10.1016/j.cell.2021.03.025)
</details>

<details>
  <summary>Kapitel 6: Bändigung des Zufalls</summary>

* _Mishra R_ (Editor) (2012) **Mutagenesis**. InTech, doi: [10.5772/2937](https://doi.org/10.5772/2937)

* _Muller HJ_ (1927) **Artificial Transmutation of the Gene**. Science 66:84-87, doi: [10.1126/science.66.1699.84](https://doi.org/10.1126/science.66.1699.84)

* _Calabrese EJ_ (2018) **Muller's nobel prize research and peer review**. Philosophy, Ethics, and Humanities in Medicine 13, doi: [10.1186/s13010-018-0066-z](https://doi.org/10.1186/s13010-018-0066-z)

* _Malling HV_ (2004) **History of the science of mutagenesis from a personal perspective**. Environ Mol Mutagen 44:372-386, doi: [10.1002/em.20064](https://doi.org/10.1002/em.20064)

* _Hamer L_, _DeZwaan TM_ _et&nbsp;al._ (2001) **Recent advances in large-scale transposon mutagenesis**. Current Opinion in Chemical Biology 5:67-73, doi: [10.1016/s1367-5931(00)00162-9](https://doi.org/10.1016/s1367-5931(00)00162-9)

* _Murray MJ_ & _Todd WA_ (1972) **Registration of todd's mitcham peppermint 1 (reg. no. 1)**. Crop Sci 12:128-128, doi: [10.2135/cropsci1972.0011183x001200010056x](https://doi.org/10.2135/cropsci1972.0011183x001200010056x)

* _Hutchison C_, _Phillips S_ _et&nbsp;al._ (1978) **Mutagenesis at a specific position in a DNA sequence.** Journal of Biological Chemistry 253:6551-6560, doi: [10.1016/s0021-9258(19)46967-6](https://doi.org/10.1016/s0021-9258(19)46967-6)
</details>

<details>
  <summary>Kapitel 7: Dolly war kein Clown</summary>

* _Booth W_, _Smith CF_ _et&nbsp;al._ (2012) **Facultative parthenogenesis discovered in wild vertebrates**. Biol Lett 8:983-985, doi: [10.1098/rsbl.2012.0666](https://doi.org/10.1098/rsbl.2012.0666)

* _Campbell KHS_, _McWhir J_ _et&nbsp;al._ (1996) **Sheep cloned by nuclear transfer from a cultured cell line**. Nature 380:64-66, doi: [10.1038/380064a0](https://doi.org/10.1038/380064a0)

* _Manafi M_ (Editor) (2011) **Artificial Insemination in Farm Animals**. InTech, doi: [10.5772/713](https://doi.org/10.5772/713)

* _Keefer CL_ (2015) **Artificial cloning of domestic animals**. Proc Natl Acad Sci USA 112:8874-8878, doi: [10.1073/pnas.1501718112](https://doi.org/10.1073/pnas.1501718112)

* _Willadsen SM_ (1979) **A method for culture of micromanipulated sheep embryos and its use to produce monozygotic twins**. Nature 277:298-300, doi: [10.1038/277298a0](https://doi.org/10.1038/277298a0)

* _Willadsen SM_ & _Polge C_ (1981) **Attempts to produce monozygotic quadruplets in cattle by blastomere separation**. Veterinary Record 108:211-213, doi: [10.1136/vr.108.10.211](https://doi.org/10.1136/vr.108.10.211)

* _Willadsen SM_ (1986) **Nuclear transplantation in sheep embryos**. Nature 320:63-65, doi: [10.1038/320063a0](https://doi.org/10.1038/320063a0)

&#8212; (1989) **Cloning of sheep and cow embryos**. Genome 31:956-962, doi: [10.1139/g89-167](https://doi.org/10.1139/g89-167)

* _Hwang WS_ (2004) **Evidence of a Pluripotent Human Embryonic Stem Cell Line Derived from a Cloned Blastocyst**. Science 303:1669-1674, doi: [10.1126/science.1094515](https://doi.org/10.1126/science.1094515)

&#8212; (2005) **Patient-Specific Embryonic Stem Cells Derived from Human SCNT Blastocysts**. Science 308:1777-1783, doi: [10.1126/science.1112286](https://doi.org/10.1126/science.1112286)

* _French AJ_, _Adams CA_ _et&nbsp;al._ (2008) **Development of Human Cloned Blastocysts Following Somatic Cell Nuclear Transfer with Adult Fibroblasts**. Stem Cells 26:485-493, doi: [10.1634/stemcells.2007-0252](https://doi.org/10.1634/stemcells.2007-0252)

* _Noggle S_, _Fung HL_ _et&nbsp;al._ (2011) **Human oocytes reprogram somatic cells to a pluripotent state**. Nature 478:70-75, doi: [10.1038/nature10397](https://doi.org/10.1038/nature10397)

* _Trounson A_ & _DeWitt ND_ (2013) **Pluripotent Stem Cells from Cloned Human Embryos: Success at Long Last**. Cell Stem Cell 12:636-638, doi: [10.1016/j.stem.2013.05.022](https://doi.org/10.1016/j.stem.2013.05.022)

* _Chung YG_, _Eum JH_ _et&nbsp;al._ (2014) **Human Somatic Cell Nuclear Transfer Using Adult Cells**. Cell Stem Cell 14:777-780, doi: [10.1016/j.stem.2014.03.015](https://doi.org/10.1016/j.stem.2014.03.015)

* _Liu Z_, _Cai Y_ _et&nbsp;al._ (2018) **Cloning of Macaque Monkeys by Somatic Cell Nuclear Transfer**. Cell 172:881-887.e7, doi: [10.1016/j.cell.2018.01.020](https://doi.org/10.1016/j.cell.2018.01.020)

* _Takahashi K_ & _Yamanaka S_ (2006) **Induction of Pluripotent Stem Cells from Mouse Embryonic and Adult Fibroblast Cultures by Defined Factors**. Cell 126:663-676, doi: [10.1016/j.cell.2006.07.024](https://doi.org/10.1016/j.cell.2006.07.024)
</details>

<details>
  <summary>Kapitel 8: Unser DNA als Glaskugel</summary>

* _Zerjal T_, _Xue Y_ _et&nbsp;al._ (2003) **The Genetic Legacy of the Mongols**. The American Journal of Human Genetics 72:717-721, doi: [10.1086/367774](https://doi.org/10.1086/367774)

* _Rawi SA_, _Louvet-Vallee S_ _et&nbsp;al._ (2011) **Postfertilization Autophagy of Sperm Organelles Prevents Paternal Mitochondrial DNA Transmission**. Science 334:1144-1147, doi: [10.1126/science.1211878](https://doi.org/10.1126/science.1211878)

* _Hentschel MW_ & _Pötzl NF_ (1986), **Die position der RAF hat sich verbessert**. Zugegriffen unter [Internetlink](https://www.spiegel.de/spiegel/print/d-13519259.html), zugegriffen am 15/01/21

* _Editorial_ (1912) **The Precipitin Reaction**. Nature 88:566-566, doi: [10.1038/088566a0](https://doi.org/10.1038/088566a0)

* _Tautz D_ & _Renz M_ (1984) **Simple sequences are ubiquitous repetitive components of eukaryotic genomes**. Nucleic Acids Research 12:4127-4138, doi: [10.1093/nar/12.10.4127](https://doi.org/10.1093/nar/12.10.4127)

* _Tautz D_, _Trick M_ & _Dover GA_ (1986) **Cryptic simplicity in DNA is a major source of genetic variation**. Nature 322:652-656, doi: [10.1038/322652a0](https://doi.org/10.1038/322652a0)

* _Jeffreys AJ_, _Wilson V_ & _Thein SL_ (1985) **Hypervariable 'minisatellite' regions in human DNA**. Nature 314:67-73, doi: [10.1038/314067a0](https://doi.org/10.1038/314067a0)

* _Hagelberg E_, _Gray IC_ & _Jeffreys AJ_ (1991) **Identification of the skeletal remains of a murder victim by DNA analysis**. Nature 352:427-429, doi: [10.1038/352427a0](https://doi.org/10.1038/352427a0)

* _Gill P_, _Jeffreys AJ_ & _Werrett DJ_ (1985) **Forensic application of DNA `fingerprints'**. Nature 318:577-579, doi: [10.1038/318577a0](https://doi.org/10.1038/318577a0)

* _Butler JM_ & _Hill CR_ (2012) **Biology and Genetics of New Autosomal STR Loci Useful for Forensic DNA Analysis**. Forensic Science Review 24:15-26

* _Hill J_ (2013) **Biology and Genetics of New Autosomal STR Loci Useful for Forensic DNA Analysis.** In: Forensic DNA Analysis, 181-198, CRC Press, doi: [10.1201/b15361-13](https://doi.org/10.1201/b15361-13)

* _Sullivan K_, _Mannucci A_ _et&nbsp;al._ (1993) **A rapid and quantitative DNA sex test: Fluorescence-based PCR analysis of X-Y homologous gene amelogenin**. BioTechniques 15:636-641

* _Weir BS_ (2007) **The rarity of DNA profiles**. The Annals of Applied Statistics 1:358-370, doi: [10.1214/07-aoas128](https://doi.org/10.1214/07-aoas128)

* _Hares DR_ (2015) **Selection and implementation of expanded CODIS core loci in the United States**. Forensic Science International: Genetics 17:33-34, doi: [10.1016/j.fsigen.2015.03.006](https://doi.org/10.1016/j.fsigen.2015.03.006)

* _Gill P_, _Ivanov PL_ _et&nbsp;al._ (1994) **Identification of the remains of the Romanov family by DNA analysis**. Nature Genetics 6:130-135, doi: [10.1038/ng0294-130](https://doi.org/10.1038/ng0294-130)

* _Wambaugh J_ (1989) **The blooding**. Bantam Books, New York, ISBN 978-0-553-76330-0

* _Erlich Y_, _Shor T_ _et&nbsp;al._ (2018) **Identity inference of genomic data using long-range familial searches**. Science 362:690-694, doi: [10.1126/science.aau4832](https://doi.org/10.1126/science.aau4832)

* _Kaiser J_ (2018) **We will find you: DNA search used to nab Golden State Killer can home in on about 60% of white Americans**. Science doi: [10.1126/science.aav7021](https://doi.org/10.1126/science.aav7021)

* _M'charek A_, _Toom V_ & _Jong L_ (2020) **The Trouble with Race in Forensic Identification**. Science, Technology, & Human Values Human Values 45:804-828, doi: [10.1177/0162243919899467](https://doi.org/10.1177/0162243919899467)

* _Pääbo S_ (1985) **Molecular cloning of Ancient Egyptian mummy DNA**. Nature 314:644-645, doi: [10.1038/314644a0](https://doi.org/10.1038/314644a0)

* _Pauling L_, _Zuckerkandl E_ _et&nbsp;al._ (1963) **Chemical Paleogenetics. Molecular „Restoration Studies“ of Extinct Forms of Life.** Acta Chem Scand 17 supl.:9-16, doi: [10.3891/acta.chem.scand.17s-0009](https://doi.org/10.3891/acta.chem.scand.17s-0009)

* _Green RE_, _Krause J_ _et&nbsp;al._ (2010) **A Draft Sequence of the Neandertal Genome**. Science 328:710-722, doi: [10.1126/science.1188021](https://doi.org/10.1126/science.1188021)

* _Reich D_, _Green RE_ _et&nbsp;al._ (2010) **Genetic history of an archaic hominin group from Denisova Cave in Siberia**. Nature 468:1053-1060, doi: [10.1038/nature09710](https://doi.org/10.1038/nature09710)

* _Prüfer K_, _Racimo F_ _et&nbsp;al._ (2013) **The complete genome sequence of a Neanderthal from the Altai Mountains**. Nature 505:43-49, doi: [10.1038/nature12886](https://doi.org/10.1038/nature12886)

* _Krause J_ (2019) **Die Reise unserer Gene eine Geschichte über uns und unsere Vorfahren**. Propyläen, Berlin, ISBN 3549100027

* _Peeters S_ & _Zwart H_ (2020) **Neanderthals as familiar strangers and the human spark: How the `golden years' of Neanderthal research reopen the question of human uniqueness**. HPLS 42, doi: [10.1007/s40656-020-00327-w](https://doi.org/10.1007/s40656-020-00327-w)

* _Slon V_, _Mafessoni F_ _et&nbsp;al._ (2018) **The genome of the offspring of a Neanderthal mother and a Denisovan father**. Nature 561:113-116, doi: [10.1038/s41586-018-0455-x](https://doi.org/10.1038/s41586-018-0455-x)

* _Hublin JJ_, _Ben-Ncer A_ _et&nbsp;al._ (2017) **New fossils from Jebel Irhoud, Morocco and the pan-African origin of Homo sapiens**. Nature 546:289-292, doi: [10.1038/nature22336](https://doi.org/10.1038/nature22336)

* _Richter D_, _Grün R_ _et&nbsp;al._ (2017) **The age of the hominin fossils from Jebel Irhoud, Morocco, and the origins of the Middle Stone Age**. Nature 546:293-296, doi: [10.1038/nature22335](https://doi.org/10.1038/nature22335)

* _Bergström A_, _Stringer C_ _et&nbsp;al._ (2021) **Origins of modern human ancestry**. Nature 590:229-237, doi: [10.1038/s41586-021-03244-5](https://doi.org/10.1038/s41586-021-03244-5)

* _Rogers AR_, _Harris NS_ & _Achenbach AA_ (2020) **Neanderthal-Denisovan ancestors interbred with a distantly related hominin**. Sci Adv 6:eaay5483, doi: [10.1126/sciadv.aay5483](https://doi.org/10.1126/sciadv.aay5483)

* _Gibbons A_ (2020) **Mysterious `ghost' populations had multiple trysts with human ancestors**. Science doi: [10.1126/science.abb3777](https://doi.org/10.1126/science.abb3777)

* _Price M_ (2020) **Africans, too, carry Neanderthal genetic legacy**. Science 367:497-497, doi: [10.1126/science.367.6477.497](https://doi.org/10.1126/science.367.6477.497)

* _Kanitz R_, _Guillot EG_ _et&nbsp;al._ (2018) **Complex genetic patterns in human arise from a simple range-expansion model over continental landmasses**. PLoS ONE 13:e0192460, doi: [10.1371/journal.pone.0192460](https://doi.org/10.1371/journal.pone.0192460)

* _Noonan JP_, _Coop G_ _et&nbsp;al._ (2006) **Sequencing and Analysis of Neanderthal Genomic DNA**. Science 314:1113-1118, doi: [10.1126/science.1131412](https://doi.org/10.1126/science.1131412)

* _Jeong C_, _Alkorta-Aranburu G_ _et&nbsp;al._ (2014) **Admixture facilitates genetic adaptations to high altitude in Tibet**. Nat Commun 5, doi: [10.1038/ncomms4281](https://doi.org/10.1038/ncomms4281)

* _Miao B_, _Wang Z_ & _Li Y_ (2017) **Genomic Analysis Reveals Hypoxia Adaptation in the Tibetan Mastiff by Introgression of the Grey Wolf from the Tibetan Plateau**. Mol Biol Evol 34:734-743, doi: [10.1093/molbev/msw274](https://doi.org/10.1093/molbev/msw274)

* _Liu X_, _Zhang Y_ _et&nbsp;al._ (2019) **EPAS1 Gain-of-Function Mutation Contributes to High-Altitude Adaptation in Tibetan Horses**. Molecular Biology and Evolution 36:2591-2603, doi: [10.1093/molbev/msz158](https://doi.org/10.1093/molbev/msz158)

* _Wang MS_, _Wang S_ _et&nbsp;al._ (2020) **Ancient Hybridization with an Unknown Population Facilitated High-Altitude Adaptation of Canids**. Molecular Biology and Evolution 37:2616-2629, doi: [10.1093/molbev/msaa113](https://doi.org/10.1093/molbev/msaa113)

* _Nédélec Y, <em>Sanz J_ _et&nbsp;al._ (2016) **Genetic Ancestry and Natural Selection Drive Population Differences in Immune Responses to Pathogens**. Cell 167:657-669.e21, doi: [10.1016/j.cell.2016.09.025](https://doi.org/10.1016/j.cell.2016.09.025)

* _Quach H_, _Rotival M_ _et&nbsp;al._ (2016) **Genetic Adaptation and Neandertal Admixture Shaped the Immune System of Human Populations**. Cell 167:643-656.e17, doi: [10.1016/j.cell.2016.09.024](https://doi.org/10.1016/j.cell.2016.09.024)

* _Enard D_ & _Petrov DA_ (2018) **Evidence that RNA Viruses Drove Adaptive Introgression between Neanderthals and Modern Humans**. Cell 175:360-371.e13, doi: [10.1016/j.cell.2018.08.034](https://doi.org/10.1016/j.cell.2018.08.034)

* _Warren M_ (2018) **Diverse genome study upends understanding of how language evolved**. Nature doi: [10.1038/d41586-018-05859-7](https://doi.org/10.1038/d41586-018-05859-7)

* _Miller W_, _Drautz DI_ _et&nbsp;al._ (2008) **Sequencing the nuclear genome of the extinct woolly mammoth**. Nature 456:387-390, doi: [10.1038/nature07446](https://doi.org/10.1038/nature07446)

* _Gaunitz C_, _Fages A_ _et&nbsp;al._ (2018) **Ancient genomes revisit the ancestry of domestic and Przewalski's horses**. Science 360:111-114, doi: [10.1126/science.aao3297](https://doi.org/10.1126/science.aao3297)

* _Ramos-Madrigal J_, _Runge AKW_ _et&nbsp;al._ (2019) **Palaeogenomic insights into the origins of French grapevine diversity**. Nat Plants 5:595-603, doi: [10.1038/s41477-019-0437-5](https://doi.org/10.1038/s41477-019-0437-5)

* _Kistler L_, _Maezumi SY_ _et&nbsp;al._ (2018) **Multiproxy evidence highlights a complex evolutionary legacy of maize in South America**. Science 362:1309-1313, doi: [10.1126/science.aav0207](https://doi.org/10.1126/science.aav0207)

* _Gros-Balthazard M_, _Flowers JM_ _et&nbsp;al._ (2021) **The genomes of ancient date palms germinated from 2,000 y old seeds**. Proc Natl Acad Sci USA 118:e2025337118, doi: [10.1073/pnas.2025337118](https://doi.org/10.1073/pnas.2025337118)

* _Beal WJ_ (1905) **The Vitality of Seeds**. Botanical Gazette 40:140-143

* _Telewski FW_ & _Zeevaart JAD_ (2002) **The 120-yr period for Dr. Beals seed viability experiment**. American Journal of Botany 89:1285-1288, doi: [10.3732/ajb.89.8.1285](https://doi.org/10.3732/ajb.89.8.1285)

* _Buniello A_, _MacArthur JAL_ _et&nbsp;al._ (2018) **The NHGRI-EBI GWAS Catalog of published genome-wide association studies, targeted arrays and summary statistics 2019**. Nucleic Acids Research 47:D1005-D1012, doi: [10.1093/nar/gky1120](https://doi.org/10.1093/nar/gky1120)

* _Klein RJ_ (2005) **Complement Factor H Polymorphism in Age-Related Macular Degeneration**. Science 308:385-389, doi: [10.1126/science.1109557](https://doi.org/10.1126/science.1109557)

* _Boutin TS_, _Charteris DG_ _et&nbsp;al._ (2019) **Insights into the genetic basis of retinal detachment**. Human Molecular Genetics 29:689-702, doi: [10.1093/hmg/ddz294](https://doi.org/10.1093/hmg/ddz294)

* _Watanabe K_, _Stringer S_ _et&nbsp;al._ (2019) **A global overview of pleiotropy and genetic architecture in complex traits**. Nat Genet 51:1339-1348, doi: [10.1038/s41588-019-0481-0](https://doi.org/10.1038/s41588-019-0481-0)

* _Mullins N_, _Forstner AJ_ _et&nbsp;al._ (2021) **Genome-wide association study of more than 40,000 bipolar disorder cases provides new insights into the underlying biology**. Nat Genet 53:817-829, doi: [10.1038/s41588-021-00857-4](https://doi.org/10.1038/s41588-021-00857-4)

* _Gallagher MD_ & _Chen-Plotkin AS_ (2018) **The Post-GWAS Era: From Association to Function**. The American Journal of Human Genetics 102:717-730, doi: [10.1016/j.ajhg.2018.04.002](https://doi.org/10.1016/j.ajhg.2018.04.002)

* _Cuervo-Alarcon L_, _Arend M_ _et&nbsp;al._ (2021) **A candidate gene association analysis identifies SNPs potentially involved in drought tolerance in European beech (Fagus sylvatica L.)**. Scientific Reports 11, doi: [10.1038/s41598-021-81594-w](https://doi.org/10.1038/s41598-021-81594-w)

* _Zhang W_, _Reeves GR_ & _Tautz D_ (2021) **Testing Implications of the Omnigenic Model for the Genetic Analysis of Loci Identified through Genome-wide Association**. Current Biology 31:1092-1098.e6, doi: [10.1016/j.cub.2020.12.023](https://doi.org/10.1016/j.cub.2020.12.023)

* _Galton F_ (1875) **The History of Twins, As a Criterion of the Relative Powers of Nature and Nurture**. Fraser’s Magazine 12:566-76

* _Bouchard TJ_ (1984) **Twins Reared Together and Apart: What They Tell Us About Human Diversity.** In: Individuality and Determinism, 147-184, Springer US, doi: [10.1007/978-1-4615-9379-9_7](https://doi.org/10.1007/978-1-4615-9379-9_7)

* _Crouch DJM_, _Winney B_ _et&nbsp;al._ (2018) **Genetics of the human face: Identification of large-effect single gene variants**. Proc Natl Acad Sci USA 115:E676-E685, doi: [10.1073/pnas.1708207114](https://doi.org/10.1073/pnas.1708207114)

* _Galton F_ (1879) **Composite Portraits, Made by Combining Those of Many Different Persons Into a Single Resultant Figure.** The Journal of the Anthropological Institute of Great Britain and Ireland 8:132, doi: [10.2307/2841021](https://doi.org/10.2307/2841021)

* _Xiong Z_, _Dankova G_ _et&nbsp;al._ (2019) **Novel genetic loci affecting facial shape variation in humans. eLife** 8, doi: [10.7554/elife.49898](https://doi.org/10.7554/elife.49898)

* _Gurovich Y_, _Hanani Y_ _et&nbsp;al._ (2019) **Identifying facial phenotypes of genetic disorders using deep learning**. Nat Med 25:60-64, doi: [10.1038/s41591-018-0279-0](https://doi.org/10.1038/s41591-018-0279-0)

* _Davenport GC_ & _Davenport CB_ (1907) **Heredity of Eye Color in Man**. Science 26:589-592, doi: [10.1126/science.26.670.589-b](https://doi.org/10.1126/science.26.670.589-b)

* _Simcoe M_, _Valdes A_ _et&nbsp;al._ (2021) **Genome-wide association study in almost 195,000 individuals identifies 50 previously unidentified genetic loci for eye color**. Sci Adv 7:eabd1239, doi: [10.1126/sciadv.abd1239](https://doi.org/10.1126/sciadv.abd1239)

* _Townsend P_, **Dna's twist to the right is not to be meddled with, so let's lose the lefties**. Zugegriffen unter [Internetlink](https://magazine.ucsc.edu/2020/02/beating-crime-by-a-hair/), zugegriffen am 19/03/21
</details>

<details>
  <summary>Kapitel 9: Künstliche Gene und Organismen</summary>

* _Ro DK_, _Paradise EM_ _et&nbsp;al._ (2006) **Production of the antimalarial drug precursor artemisinic acid in engineered yeast**. Nature 440:940-943, doi: [10.1038/nature04640](https://doi.org/10.1038/nature04640)

* _Hommel M_ (2008) **The future of artemisinins: Natural, synthetic or recombinant?** Journal of Biology 7:38, doi: [10.1186/jbiol101](https://doi.org/10.1186/jbiol101)

* _Peplow M_ (2016) **Synthetic biology's first malaria drug meets market resistance**. Nature 530:389-390, doi: [10.1038/530390a](https://doi.org/10.1038/530390a)

* _Westfall PJ_, _Pitera DJ_ _et&nbsp;al._ (2012) **Production of amorphadiene in yeast, and its conversion to dihydroartemisinic acid, precursor to the antimalarial agent artemisinin**. Proc Natl Acad Sci USA 109:E111-E118, doi: [10.1073/pnas.1110740109](https://doi.org/10.1073/pnas.1110740109)

* _Paddon CJ_, _Westfall PJ_ _et&nbsp;al._ (2013) **High-level semi-synthetic production of the potent antimalarial artemisinin**. Nature 496:528-532, doi: [10.1038/nature12051](https://doi.org/10.1038/nature12051)

* _Kohman RE_, _Kunjapur AM_ _et&nbsp;al._ (2018) **From Designing the Molecules of Life to Designing Life: Future Applications Derived from Advances in DNA Technologies**. Angew Chem Int Ed 57:4313-4328, doi: [10.1002/anie.201707976](https://doi.org/10.1002/anie.201707976)

* _Sturtevant AH_ (1923) **Inheritance of Direction of Coilling in Limnaea**. Science 58:269-270, doi: [10.1126/science.58.1501.269](https://doi.org/10.1126/science.58.1501.269)

* _Wang L_, _Xie J_ & _Schultz PG_ (2006) **Expanding the Genetic Code**. Annu Rev Biophys Biomol Struct 35:225-249, doi: [10.1146/annurev.biophys.35.101105.121507](https://doi.org/10.1146/annurev.biophys.35.101105.121507)

* _Ma NJ_ & _Isaacs FJ_ (2016) **Genomic Recoding Broadly Obstructs the Propagation of Horizontally Transferred Genetic Elements**. Cell Systems 3:199-207, doi: [10.1016/j.cels.2016.06.009](https://doi.org/10.1016/j.cels.2016.06.009)

* _de la Torre D_ & _Chin JW_ (2020) **Reprogramming the genetic code**. Nat Rev Genet 22:169-184, doi: [10.1038/s41576-020-00307-7](https://doi.org/10.1038/s41576-020-00307-7)

* _Dunkelmann DL_, _Willis JCW_ _et&nbsp;al._ (2020) **Engineered triply orthogonal pyrrolysyl-tRNA synthetase/tRNA pairs enable the genetic encoding of three distinct non-canonical amino acids**. Nat Chem 12:535-544, doi: [10.1038/s41557-020-0472-x](https://doi.org/10.1038/s41557-020-0472-x)

* _Neumann H_, _Wang K_ _et&nbsp;al._ (2010) **Encoding multiple unnatural amino acids via evolution of a quadruplet-decoding ribosome**. Nature 464:441-444, doi: [10.1038/nature08817](https://doi.org/10.1038/nature08817)

* _Chin JW_ (2017) **Expanding and reprogramming the genetic code**. Nature 550:53-60, doi: [10.1038/nature24031](https://doi.org/10.1038/nature24031)

* _Khorana H_, _Agarwal K_ _et&nbsp;al._ (1972) **Total synthesis of the structural gene for an alanine transfer ribonucleic acid from yeast**. Journal of Molecular Biology 72:209-217, doi: [10.1016/0022-2836(72)90146-5](https://doi.org/10.1016/0022-2836(72)90146-5)

* _Gibson DG_, _Glass JI_ _et&nbsp;al._ (2010) **Creation of a Bacterial Cell Controlled by a Chemically Synthesized Genome**. Science 329:52-56, doi: [10.1126/science.1190719](https://doi.org/10.1126/science.1190719)

* _Annaluru N_, _Muller H_ _et&nbsp;al._ (2014) **Total Synthesis of a Functional Designer Eukaryotic Chromosome**. Science 344:55-58, doi: [10.1126/science.1249252](https://doi.org/10.1126/science.1249252)

* _Wimmer E_ (2006) **The test-tube synthesis of a chemical called poliovirus. EMBO** Rep reports 7, doi: [10.1038/sj.embor.7400728](https://doi.org/10.1038/sj.embor.7400728)

* _Lentzos F_ (2012) **Synthetic biology, security and governance**. BioSocieties 7:339-351, doi: [10.1057/biosoc.2012.28](https://doi.org/10.1057/biosoc.2012.28)

* _Abeli T_, _Rocchetti GA_ _et&nbsp;al._ (2021) **Seventeen `extinct' plant species back to conservation attention in Europe**. Nature Plants 7:282-286, doi: [10.1038/s41477-021-00878-1](https://doi.org/10.1038/s41477-021-00878-1)

* _Folch J_, _Cocero M_ _et&nbsp;al._ (2009) **First birth of an animal from an extinct subspecies (Capra pyrenaica pyrenaica) by cloning**. Theriogenology 71:1026-1034, doi: [10.1016/j.theriogenology.2008.11.005](https://doi.org/10.1016/j.theriogenology.2008.11.005)

* _Yamagata K_, _Nagai K_ _et&nbsp;al._ (2019) **Signs of biological activities of 28,000-year-old mammoth nuclei in mouse oocytes visualized by live-cell imaging**. Sci Rep 9, doi: [10.1038/s41598-019-40546-1](https://doi.org/10.1038/s41598-019-40546-1)

* _Griffin DK_, _Larkin DM_ & _O'Connor RE_ (2020) **Time lapse: A glimpse into prehistoric genomics**. European Journal of Medical Genetics 63:103640, doi: [10.1016/j.ejmg.2019.03.004](https://doi.org/10.1016/j.ejmg.2019.03.004)

* _Wright DWM_ (2018) **Cloning animals for tourism in the year 2070**. Futures 95:58-75, doi: [10.1016/j.futures.2017.10.002](https://doi.org/10.1016/j.futures.2017.10.002)
</details>

<details>
  <summary>Kapitel 10: Künstliche Intelligenz und echte Erkenntnisse</summary>

* _Bycroft C_, _Freeman C_ _et&nbsp;al._ (2018) **The UK Biobank resource with deep phenotyping and genomic data**. Nature 562:203-209, doi: [10.1038/s41586-018-0579-z](https://doi.org/10.1038/s41586-018-0579-z)

* _Pearce F_ (2008) **Ozone hole? What ozone hole?** New Scientist 199:46-47

* _Farman JC_, _Gardiner BG_ & _Shanklin JD_ (1985) **Large losses of total ozone in Antarctica reveal seasonal ClOx/NOx interaction**. Nature 315:207-210, doi: [10.1038/315207a0](https://doi.org/10.1038/315207a0)

* _Seeland M_, _Rzanny M_ _et&nbsp;al._ (2019) **Image-based classification of plant genus and family for trained and untrained plant species**. BMC Bioinformatics 20, doi: [10.1186/s12859-018-2474-x](https://doi.org/10.1186/s12859-018-2474-x)

* _Navarro FCP_, _Mohsen H_ _et&nbsp;al._ (2019) **Genomics and data science: An application within an umbrella**. Genome Biol 20, doi: [10.1186/s13059-019-1724-1](https://doi.org/10.1186/s13059-019-1724-1)

* _Jordan MI_ & _Mitchell TM_ (2015) **Machine learning: Trends, perspectives, and prospects**. Science 349:255-260, doi: [10.1126/science.aaa8415](https://doi.org/10.1126/science.aaa8415)

* _Silver D_, _Schrittwieser J_ _et&nbsp;al._ (2017) **Mastering the game of Go without human knowledge**. Nature 550:354-359, doi: [10.1038/nature24270](https://doi.org/10.1038/nature24270)

* _Silver D_, _Hubert T_ _et&nbsp;al._ (2018) **A general reinforcement learning algorithm that masters chess, shogi, and Go through self-play**. Science 362:1140-1144, doi: [10.1126/science.aar6404](https://doi.org/10.1126/science.aar6404)
</details>

<details>
  <summary>Kapitel 11: Gentechnik und Umweltschutz – geht das?</summary>

* _Crutzen P_ & _Stoermer E_ (2000) **The Anthropocene**. Global Change Newsl 41:17-18

* _Pimentel D_ (2008) **Food, energy, and society**. CRC Press, Boca Raton, FL, ISBN 1420046675

* _Hill J_, _Nelson E_ _et&nbsp;al._ (2006) **Environmental, economic, and energetic costs and benefits of biodiesel and ethanol biofuels**. Proc Natl Acad Sci USA 103:11206-11210, doi: [10.1073/pnas.0604600103](https://doi.org/10.1073/pnas.0604600103)

* _Yue M_, _Lambert H_ _et&nbsp;al._ (2021) **Hydrogen energy systems: A critical review of technologies, applications, trends and challenges**. Renewable and Sustainable Energy Reviews 146:111180, doi: [10.1016/j.rser.2021.111180](https://doi.org/10.1016/j.rser.2021.111180)

* _Appel J_, _Hueren V_ _et&nbsp;al._ (2020) **Cyanobacterial in vivo solar hydrogen production using a photosystem I-hydrogenase (PsaD-HoxYH) fusion complex**. Nat Energy 5:458-467, doi: [10.1038/s41560-020-0609-6](https://doi.org/10.1038/s41560-020-0609-6)

* _Wünschiers R_ (2015) **Making-of Synthetic Biology: The European CyanoFactory Research Consortium.** In: Ethics of Science and Technology Assessment, 55-72, Springer International Publishing, doi: [10.1007/978-3-319-21088-9_3](https://doi.org/10.1007/978-3-319-21088-9_3)

* _Katayama N_, _Iijima H_ & _Osanai T_ (2018) **Production of Bioplastic Compounds by Genetically Manipulated and Metabolic Engineered Cyanobacteria.** In: Synthetic Biology of Cyanobacteria, 155-169, Springer Singapore, doi: [10.1007/978-981-13-0854-3_7](https://doi.org/10.1007/978-981-13-0854-3_7)

* _Heidorn T_, _Camsund D_ _et&nbsp;al._ (2011) **Synthetic Biology in Cyanobacteria.** In: Methods in Enzymology, 539-579, Elsevier, doi: [10.1016/b978-0-12-385075-1.00024-x](https://doi.org/10.1016/b978-0-12-385075-1.00024-x)

* _DiCarlo JE_, _Chavez A_ _et&nbsp;al._ (2015) **Safeguarding CRISPR-Cas9 gene drives in yeast**. Nat Biotechnol 33:1250-1255, doi: [10.1038/nbt.3412](https://doi.org/10.1038/nbt.3412)

* _Gantz VM_, _Jasinskiene N_ _et&nbsp;al._ (2015) **Highly efficient Cas9-mediated gene drive for population modification of the malaria vector mosquito Anopheles stephensi**. Proc Natl Acad Sci USA 112:E6736-E6743, doi: [10.1073/pnas.1521077112](https://doi.org/10.1073/pnas.1521077112)

* _Simon S_, _Otto M_ & _Engelhard M_ (2018) **Synthetic gene drive: Between continuity and novelty**. EMBO Reports 19, doi: [10.15252/embr.201845760](https://doi.org/10.15252/embr.201845760)

* _Tanaka H_, _Stone HA_ & _Nelson DR_ (2017) **Spatial gene drives and pushed genetic waves**. Proc Natl Acad Sci USA 114:8452-8457, doi: [10.1073/pnas.1705868114](https://doi.org/10.1073/pnas.1705868114)

* _Esvelt KM_ & _Gemmell NJ_ (2017) **Conservation demands safe gene drive**. PLoS Biology 15:e2003850, doi: [10.1371/journal.pbio.2003850](https://doi.org/10.1371/journal.pbio.2003850)

* _Jones MS_, _Delborne JA_ _et&nbsp;al._ (2019) **Does the U.S. public support using gene drives in agriculture? And what do they want to know?** Sci Adv 5:eaau8462, doi: [10.1126/sciadv.aau8462](https://doi.org/10.1126/sciadv.aau8462)

* _van Kleunen M_, _Dawson W_ _et&nbsp;al._ (2015) **Global exchange and accumulation of non-native plants**. Nature 525:100-103, doi: [10.1038/nature14910](https://doi.org/10.1038/nature14910)

* _Schrimpf A_ (2014) **Invasive Chinese mitten crab (Eriocheir sinensis) transmits crayfish plague pathogen (Aphanomyces astaci)**. AI 9:203-209, doi: [10.3391/ai.2014.9.2.09](https://doi.org/10.3391/ai.2014.9.2.09)

* _Phillips BL_, _Brown GP_ _et&nbsp;al._ (2006) **Invasion and the evolution of speed in toads**. Nature 439:803-803, doi: [10.1038/439803a](https://doi.org/10.1038/439803a)

* _Wisely SM_, _Ryder OA_ _et&nbsp;al._ (2015) **A Road Map for 21st Century Genetic Restoration: Gene Pool Enrichment of the Black-Footed Ferret**. JHERED 106:581-592, doi: [10.1093/jhered/esv041](https://doi.org/10.1093/jhered/esv041)

* _Breed MF_, _Harrison PA_ _et&nbsp;al._ (2019) **The potential of genomics for restoring ecosystems and biodiversity**. Nat Rev Genet 20:615-628, doi: [10.1038/s41576-019-0152-0](https://doi.org/10.1038/s41576-019-0152-0)

* _Shapiro B_ (2016) **Pathways to de-extinction: How close can we get to resurrection of an extinct species?** Funct Ecol 31:996-1002, doi: [10.1111/1365-2435.12705](https://doi.org/10.1111/1365-2435.12705)

* _Novak B_ (2018) **De-Extinction**. Genes 9:548, doi: [10.3390/genes9110548](https://doi.org/10.3390/genes9110548)

* _Powell WA_, _Newhouse AE_ & _Coffey V_ (2019) **Developing Blight-Tolerant American Chestnut Trees**. Cold Spring Harb Perspect Biol 11:a034587, doi: [10.1101/cshperspect.a034587](https://doi.org/10.1101/cshperspect.a034587)

* _Polin LD_, _Liang H_ _et&nbsp;al._ (2005) **Agrobacterium-mediated transformation of American chestnut (Castanea dentata (Marsh.) Borkh.) somatic embryos**. Plant Cell Tiss Organ Cult 84:69-79, doi: [10.1007/s11240-005-9002-1](https://doi.org/10.1007/s11240-005-9002-1)

* _Power ME_ (2021) **Synthetic threads through the web of life**. Proc Natl Acad Sci USA 118:e2004833118, doi: [10.1073/pnas.2004833118](https://doi.org/10.1073/pnas.2004833118)
</details>

<details>
  <summary>Kapitel 12: Epigenetik: Darwin und Lamarck vereint</summary>

* _Heard E_ & _Martienssen RA_ (2014) **Transgenerational Epigenetic Inheritance: Myths and Mechanisms**. Cell 157:95-109, doi: [10.1016/j.cell.2014.02.045](https://doi.org/10.1016/j.cell.2014.02.045)

* _Garnier O_, _Laoueillé-Duprat S_ & _Spillane C_ (2008) **Genomic imprinting in plants**. Epigenetics 3:14-20, doi: [10.4161/epi.3.1.5554](https://doi.org/10.4161/epi.3.1.5554)

* _Gustafsson A_ (1979) **Linnaeus' Peloria: The history of a monster**. Theoret Appl Genetics 54:241-248, doi: [10.1007/bf00281206](https://doi.org/10.1007/bf00281206)

* _Cubas P_, _Vincent C_ & _Coen E_ (1999) **An epigenetic mutation responsible for natural variation in floral symmetry**. Nature 401:157-161, doi: [10.1038/43657](https://doi.org/10.1038/43657)

* _Crouse HV_ (1960) **The Controlling Element in Sex Chromosome Behavior in Sciara**. Genetics 45:1429-1443, doi: [10.1093/genetics/45.10.1429](https://doi.org/10.1093/genetics/45.10.1429)

* _McGrath J_ & _Solter D_ (1983) **Nuclear transplantation in mouse embryos**. J Exp Zool 228:355-362, doi: [10.1002/jez.1402280218](https://doi.org/10.1002/jez.1402280218)

* &#8212; (1984) **Completion of mouse embryogenesis requires both the maternal and paternal genomes**. Cell 37:179-183, doi: [10.1016/0092-8674(84)90313-1](https://doi.org/10.1016/0092-8674(84)90313-1)

* _Ehrlich M_ & _Wang R_ (1981) **5-Methylcytosine in eukaryotic DNA**. Science 212:1350-1357, doi: [10.1126/science.6262918](https://doi.org/10.1126/science.6262918)

* _Li ZK_, _Wang LY_ _et&nbsp;al._ (2018) **Generation of Bimaternal and Bipaternal Mice from Hypomethylated Haploid ESCs with Imprinting Region Deletions**. Cell Stem Cell 23:665-676.e4, doi: [10.1016/j.stem.2018.09.004](https://doi.org/10.1016/j.stem.2018.09.004)

* _Tucci V_, _Isles AR_ _et&nbsp;al._ (2019) **Genomic Imprinting and Physiological Processes in Mammals**. Cell 176:952-965, doi: [10.1016/j.cell.2019.01.043](https://doi.org/10.1016/j.cell.2019.01.043)

* _Goday C_ & _Esteban MR_ (2001) **Chromosome elimination in sciarid flies**. Bioessays 23:242-250, doi: [10.1002/1521-1878(200103)23:3<242::aid-bies1034>3.0.co;2-p](https://doi.org/10.1002/1521-1878(200103)23:3<242::aid-bies1034>3.0.co;2-p)

* _Barlow DP_, _Stöger R_ _et&nbsp;al._ (1991) **The mouse insulin-like growth factor type-2 receptor is imprinted and closely linked to the Tme locus**. Nature 349:84-87, doi: [10.1038/349084a0](https://doi.org/10.1038/349084a0)

* _Renfree MB_, _Suzuki S_ & _Kaneko-Ishino T_ (2013) **The origin and evolution of genomic imprinting and viviparity in mammals**. Phil Trans R Soc B 368:20120151, doi: [10.1098/rstb.2012.0151](https://doi.org/10.1098/rstb.2012.0151)

* _Barlow DP_ & _Bartolomei MS_ (2014) **Genomic Imprinting in Mammals**. Cold Spring Harbor Perspectives in Biology 6:a018382-a018382, doi: [10.1101/cshperspect.a018382](https://doi.org/10.1101/cshperspect.a018382)

* _Wolpert L_ (2019) **Principles of development**. Oxford University Press, Oxford, United Kingdom New York, NY, ISBN 9780192520814

* _Cleaton MA_, _Edwards CA_ & _Ferguson-Smith AC_ (2014) **Phenotypic Outcomes of Imprinted Gene Models in Mice: Elucidation of Pre- and Postnatal Functions of Imprinted Genes**. Annu Rev Genom Hum Genet 15:93-126, doi: [10.1146/annurev-genom-091212-153441](https://doi.org/10.1146/annurev-genom-091212-153441)

* _Bernstein C_ & _Bernstein H_ (2020) **Demethylation in Early Embryonic Development and Memory.** In: DNA Methylation Mechanism, IntechOpen, doi: [10.5772/intechopen.90306](https://doi.org/10.5772/intechopen.90306)

* _Fernandes J_, _Arida RM_ & _Gomez-Pinilla F_ (2017) **Physical exercise as an epigenetic modulator of brain plasticity and cognition**. Neuroscience & Biobehavioral Reviews Biobehavioral Reviews 80:443-456, doi: [10.1016/j.neubiorev.2017.06.012](https://doi.org/10.1016/j.neubiorev.2017.06.012)

* _Ingerslev LR_, _Donkin I_ _et&nbsp;al._ (2018) **Endurance training remodels sperm-borne small RNA expression and methylation at neurological gene hotspots**. Clin Epigenet 10, doi: [10.1186/s13148-018-0446-7](https://doi.org/10.1186/s13148-018-0446-7)

* _Plasschaert RN_ & _Bartolomei MS_ (2014) **Genomic imprinting in development, growth, behavior and stem cells**. Development 141:1805-1813, doi: [10.1242/dev.101428](https://doi.org/10.1242/dev.101428)

* _Ugur B_, _Chen K_ & _Bellen HJ_ (2016) **Drosophilatools and assays for the study of human diseases. Disease Models &** Mechanisms 9:235-244, doi: [10.1242/dmm.023762](https://doi.org/10.1242/dmm.023762)

* _Mirzoyan Z_, _Sollazzo M_ _et&nbsp;al._ (2019) **Drosophila melanogaster: A Model Organism to Study Cancer**. Frontiers in Genetics 10, doi: [10.3389/fgene.2019.00051](https://doi.org/10.3389/fgene.2019.00051)

* _Bygren LO_, _Kaati G_ & _Edvinsson S_ (2001) **Longevity Determined by Paternal Ancestors' Nutrition during Their Slow Growth Period**. Acta Biotheoretica 49:53-59, doi: [10.1023/a:1010241825519](https://doi.org/10.1023/a:1010241825519)

* _Kaati G_, _Bygren L_ & _Edvinsson S_ (2002) **Cardiovascular and diabetes mortality determined by nutrition during parents' and grandparents' slow growth period**. Eur J Hum Genet 10:682-688, doi: [10.1038/sj.ejhg.5200859](https://doi.org/10.1038/sj.ejhg.5200859)

* _Yoshizaki K_, _Kimura R_ _et&nbsp;al._ (2021) **Paternal age affects offspring via an epigenetic mechanism involving REST/NRSF**. EMBO Rep reports 22, doi: [10.15252/embr.202051524](https://doi.org/10.15252/embr.202051524)

* _B DD_ (1930) **Degeneration and Regeneration of the Nervous System**. Nature 125:230-231, doi: [10.1038/125230a0](https://doi.org/10.1038/125230a0)

* _Lu Y_, _Brommer B_ _et&nbsp;al._ (2020) **Reprogramming to recover youthful epigenetic information and restore vision**. Nature 588:124-129, doi: [10.1038/s41586-020-2975-4](https://doi.org/10.1038/s41586-020-2975-4)
</details>

<details>
  <summary>Kapitel 13: Gentherapien</summary>

* _Abbott A_ (1992) **German state unexpectedly approves first gene trials**. Nature 360:702-702, doi: [10.1038/360702a0](https://doi.org/10.1038/360702a0)

* _Couzin-Frankel J_ (2015) **Baby's leukemia recedes after novel cell therapy**. Science 350:731-731, doi: [10.1126/science.350.6262.731](https://doi.org/10.1126/science.350.6262.731)

* &#8212; (2020) **CRISPR takes on cancer**. Science 367:616-616, doi: [10.1126/science.367.6478.616](https://doi.org/10.1126/science.367.6478.616)

* _Li H_, _Yang Y_ _et&nbsp;al._ (2020) **Applications of genome editing technology in the targeted therapy of human diseases: Mechanisms, advances and prospects**. Sig Transduct Target Ther 5, doi: [10.1038/s41392-019-0089-y](https://doi.org/10.1038/s41392-019-0089-y)

* _Zittersteijn HA_, _Harteveld CL_ _et&nbsp;al._ (2021) **A Small Key for a Heavy Door: Genetic Therapies for the Treatment of Hemoglobinopathies**. Front Genome Ed 2, doi: [10.3389/fgeed.2020.617780](https://doi.org/10.3389/fgeed.2020.617780)

* _Frangoul H_, _Altshuler D_ _et&nbsp;al._ (2021) **CRISPR-Cas9 Gene Editing for Sickle Cell Disease and &beta;-Thalassemia**. N Engl J Med 384:252-260, doi: [10.1056/nejmoa2031054](https://doi.org/10.1056/nejmoa2031054)

* _Guo P_, _Yang J_ _et&nbsp;al._ (2019) **Therapeutic genome editing of triple-negative breast tumors using a noncationic and deformable nanolipogel**. Proc Natl Acad Sci USA 116:18295-18303, doi: [10.1073/pnas.1904697116](https://doi.org/10.1073/pnas.1904697116)

* _Subbaraman N_ (2021) **Limit on lab-grown human embryos dropped by stem-cell body**. Nature 594:18-19, doi: [10.1038/d41586-021-01423-y](https://doi.org/10.1038/d41586-021-01423-y)

* _Cowgill BO_, _Bogart LM_ _et&nbsp;al._ (2008) **Fears About HIV Transmission in Families With an HIV-Infected Parent: A Qualitative Analysis**. PEDIATRICS 122:e950-e958, doi: [10.1542/peds.2008-0390](https://doi.org/10.1542/peds.2008-0390)

* _Liu R_, _Paxton WA_ _et&nbsp;al._ (1996) **Homozygous Defect in HIV-1 Coreceptor Accounts for Resistance of Some Multiply-Exposed Individuals to HIV-1 Infection**. Cell 86:367-377, doi: [10.1016/s0092-8674(00)80110-5](https://doi.org/10.1016/s0092-8674(00)80110-5)

* _Galvani AP_ & _Slatkin M_ (2003) **Evaluating plague and smallpox as historical selective pressures for the CCR5- 32 HIV-resistance allele**. Proc Natl Acad Sci USA 100:15276-15279, doi: [10.1073/pnas.2435085100](https://doi.org/10.1073/pnas.2435085100)

* _Allers K_, _Hütter G_ _et&nbsp;al._ (2011) **Evidence for the cure of HIV infection by CCR5&Delta;32 stem cell transplantation**. Blood 117:2791-2799, doi: [10.1182/blood-2010-09-309591](https://doi.org/10.1182/blood-2010-09-309591)

* _Gupta RK_, _Abdul-Jawad S_ _et&nbsp;al._ (2019) **HIV-1 remission following CCR5&Delta;32 haematopoietic stem-cell transplantation**. Nature 568:244-248, doi: [10.1038/s41586-019-1027-4](https://doi.org/10.1038/s41586-019-1027-4)

* _Stephens JC_, _Reich DE_ _et&nbsp;al._ (1998) **Dating the Origin of the CCR5-&Delta;32 AIDS-Resistance Allele by the Coalescence of Haplotypes**. The American Journal of Human Genetics 62:1507-1515, doi: [10.1086/301867](https://doi.org/10.1086/301867)
</details>

<details>
  <summary>Kapitel 14: Wie Transhumanisten uns für die Zukunft wappnen (wollen)</summary>

* _Wünschiers R_ (2016) **Über die Modernisierung des Menschen.** In: Modernes Leben – Leben in der Moderne, 123-145, Springer Fachmedien Wiesbaden, doi: [10.1007/978-3-658-13752-6_9](https://doi.org/10.1007/978-3-658-13752-6_9)

* _Huxley J_ (1968) **Transhumanism**. Journal of Humanistic Psychology 8:73-76, doi: [10.1177/002216786800800107](https://doi.org/10.1177/002216786800800107)

* _Bostrom N_ (2005) **A history of transhumanist thought**. Journal of Evolution and Technology 14:1-25

* _Finch J_ (2011) **The ancient origins of prosthetic medicine**. The Lancet 377:548-549, doi: [10.1016/s0140-6736(11)60190-6](https://doi.org/10.1016/s0140-6736(11)60190-6)

* _Stingl K_, _Bartz-Schmidt KU_ _et&nbsp;al._ (2015) **Subretinal Visual Implant Alpha IMS - Clinical trial interim report**. Vision Research 111:149-160, doi: [10.1016/j.visres.2015.03.001](https://doi.org/10.1016/j.visres.2015.03.001)

* _Chuang AT_, _Margo CE_ & _Greenberg PB_ (2014) **Retinal implants: A systematic review**. Br J Ophthalmol 98:852-856, doi: [10.1136/bjophthalmol-2013-303708](https://doi.org/10.1136/bjophthalmol-2013-303708)

* _Zrenner E_ (2012) **Solar cells for the blind**. Nature Photon 6:344-345, doi: [10.1038/nphoton.2012.114](https://doi.org/10.1038/nphoton.2012.114)

* _Humayun MS_, _Dorn JD_ _et&nbsp;al._ (2012) **Interim Results from the International Trial of Second Sight's Visual Prosthesis**. Ophthalmology 119:779-788, doi: [10.1016/j.ophtha.2011.09.028](https://doi.org/10.1016/j.ophtha.2011.09.028)

* _Tang YP_, _Shimizu E_ _et&nbsp;al._ (1999) **Genetic enhancement of learning and memory in mice**. Nature 401:63-69, doi: [10.1038/43432](https://doi.org/10.1038/43432)

* _Huang J_, _Kang BH_ _et&nbsp;al._ (2016) **Identification of a CD4-Binding-Site Antibody to HIV that Evolved Near-Pan Neutralization Breadth**. Immunity 45:1108-1121, doi: [10.1016/j.immuni.2016.10.027](https://doi.org/10.1016/j.immuni.2016.10.027)

* _Smalley E_ (2018) **FDA warns public of dangers of DIY gene therapy**. Nat Biotechnol 36:119-120, doi: [10.1038/nbt0218-119](https://doi.org/10.1038/nbt0218-119)
</details>

<details>
  <summary>Kapitel 15: Coronavirus – Wenn »negativ« positiv ist</summary>

* _Lofgren ET_ & _Fefferman NH_ (2007) **The untapped potential of virtual game worlds to shed light on real world epidemics**. The Lancet Infectious Diseases 7:625-629, doi: [10.1016/s1473-3099(07)70212-8](https://doi.org/10.1016/s1473-3099(07)70212-8)

* _Schalk A_ & _Hawn M_ (1931) **An Apparently New Respiratory Disease of Chicks**. Journal of the American Veterinary Medical Association 78:413-422

* _Boursnell MEG_, _Brown TDK_ _et&nbsp;al._ (1987) **Completion of the Sequence of the Genome of the Coronavirus Avian Infectious Bronchitis Virus**. Journal of General Virology 68:57-77, doi: [10.1099/0022-1317-68-1-57](https://doi.org/10.1099/0022-1317-68-1-57)

* _Hamre D_ & _Procknow JJ_ (1966) **A New Virus Isolated from the Human Respiratory Tract.** Experimental Biology and Medicine 121:190-193, doi: [10.3181/00379727-121-30734](https://doi.org/10.3181/00379727-121-30734)

* _Yang YL_, _Yu JQ_ & _Huang YW_ (2020) **Swine enteric alphacoronavirus (swine acute diarrhea syndrome coronavirus): An update three years after its discovery**. Virus Research 285:198024, doi: [10.1016/j.virusres.2020.198024](https://doi.org/10.1016/j.virusres.2020.198024)

* _Zhou P_, _Fan H_ _et&nbsp;al._ (2018) **Fatal swine acute diarrhoea syndrome caused by an HKU2-related coronavirus of bat origin**. Nature 556:255-258, doi: [10.1038/s41586-018-0010-9](https://doi.org/10.1038/s41586-018-0010-9)

* _Zhu N_, _Zhang D_ _et&nbsp;al._ (2020) **A Novel Coronavirus from Patients with Pneumonia in China, 2019**. New England Journal of Medicine 382:727-733, doi: [10.1056/nejmoa2001017](https://doi.org/10.1056/nejmoa2001017)

* _Wu F_, _Zhao S_ _et&nbsp;al._ (2020) **A new coronavirus associated with human respiratory disease in China**. Nature 579:265-269, doi: [10.1038/s41586-020-2008-3](https://doi.org/10.1038/s41586-020-2008-3)

* _Boni MF_, _Lemey P_ _et&nbsp;al._ (2020) **Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic**. Nat Microbiol 5:1408-1417, doi: [10.1038/s41564-020-0771-4](https://doi.org/10.1038/s41564-020-0771-4)

* _Mallapaty S_ (2021) **After the WHO report: What's next in the search for COVID's origins**. Nature 592:337-338, doi: [10.1038/d41586-021-00877-4](https://doi.org/10.1038/d41586-021-00877-4)

* _Bloom JD_, _Chan YA_ _et&nbsp;al._ (2021) **Investigate the origins of COVID-19**. Science 372:694.1-694, doi: [10.1126/science.abj0016](https://doi.org/10.1126/science.abj0016)

* _Graham RL_ & _Baric RS_ (2020) **SARS-CoV-2: Combating Coronavirus Emergence**. Immunity 52:734-736, doi: [10.1016/j.immuni.2020.04.016](https://doi.org/10.1016/j.immuni.2020.04.016)

* _Cui J_, _Li F_ & _Shi ZL_ (2019) **Origin and evolution of pathogenic coronaviruses**. Nature Reviews Microbiology 17:181-192, doi: [10.1038/s41579-018-0118-9](https://doi.org/10.1038/s41579-018-0118-9)

* _Perlman S_ (2020) **Another Decade, Another Coronavirus**. New England Journal of Medicine 382:760-762, doi: [10.1056/nejme2001126](https://doi.org/10.1056/nejme2001126)

* _Donnelly CA_, _Malik MR_ _et&nbsp;al._ (2019) **Worldwide Reduction in MERS Cases and Deaths since 2016**. Emerging Infectious Diseases 25:1758-1760, doi: [10.3201/eid2509.190143](https://doi.org/10.3201/eid2509.190143)

* _Taubenberger JK_, _Reid AH_ _et&nbsp;al._ (2005) **Characterization of the 1918 influenza virus polymerase genes**. Nature 437:889-893, doi: [10.1038/nature04230](https://doi.org/10.1038/nature04230)

* _Zimmer SM_ & _Burke DS_ (2009) **Historical Perspective – Emergence of Influenza A (H1N1) Viruses**. N Engl J Med 361:279-285, doi: [10.1056/nejmra0904322](https://doi.org/10.1056/nejmra0904322)

* _Defoe D_ (2020) **Die Pest zu London**. LIWI, Literatur- und Wissenschaftsverlag, Göttingen, ISBN 3965423347

* _Brüssow H_ & _Brüssow L_ (2021) **Clinical evidence that the pandemic from 1889 to 1891 commonly called the russian flu might have been an earlier coronavirus pandemic**. Microbial Biotechnology 14:1860-1870, doi: [10.1111/1751-7915.13889](https://doi.org/10.1111/1751-7915.13889)

* _Imai M_, _Watanabe T_ _et&nbsp;al._ (2012) **Experimental adaptation of an influenza H5 HA confers respiratory droplet transmission to a reassortant H5 HA/H1N1 virus in ferrets**. Nature 486:420-428, doi: [10.1038/nature10831](https://doi.org/10.1038/nature10831)

* _Herfst S_, _Schrauwen EJA_ _et&nbsp;al._ (2012) **Airborne Transmission of Influenza A/H5N1 Virus Between Ferrets**. Science 336:1534-1541, doi: [10.1126/science.1213362](https://doi.org/10.1126/science.1213362)

* _Brown D_ (2013) **Inferno: A novel**. Doubleday, New York, ISBN 978-0-385-53785-8

* _Gibb R_, _Redding DW_ _et&nbsp;al._ (2020) **Zoonotic host diversity increases in human-dominated ecosystems**. Nature 584:398-402, doi: [10.1038/s41586-020-2562-8](https://doi.org/10.1038/s41586-020-2562-8)

* _Johnson CK_, _Hitchens PL_ _et&nbsp;al._ (2020) **Global shifts in mammalian population trends reveal key predictors of virus spillover risk**. Proceedings of the Royal Society B: Biological Sciences 287:20192736, doi: [10.1098/rspb.2019.2736](https://doi.org/10.1098/rspb.2019.2736)

* _Irving AT_, _Ahn M_ _et&nbsp;al._ (2021) **Lessons from the host defences of bats, a unique viral reservoir**. Nature 589:363-370, doi: [10.1038/s41586-020-03128-0](https://doi.org/10.1038/s41586-020-03128-0)

* _Miller MR_, _McMinn RJ_ _et&nbsp;al._ (2016) **Broad and Temperature Independent Replication Potential of Filoviruses on Cells Derived From Old and New World Bat Species**. J Infect Dis 214:S297-S302, doi: [10.1093/infdis/jiw199](https://doi.org/10.1093/infdis/jiw199)

* _Hayman DTS_ (2019) **Bat tolerance to viral infections**. Nat Microbiol 4:728-729, doi: [10.1038/s41564-019-0430-9](https://doi.org/10.1038/s41564-019-0430-9)

* _Austad S_ (2010) **Methusalehs Zoo: How Nature provides us with Clues for Extending Human Health Span**. Journal of Comparative Pathology 142:S10-S21, doi: [10.1016/j.jcpa.2009.10.024](https://doi.org/10.1016/j.jcpa.2009.10.024)

* _Gloza-Rausch F_, _Ipsen A_ _et&nbsp;al._ (2008) **Detection and Prevalence Patterns of Group I Coronaviruses in Bats, Northern Germany**. Emerg Infect Dis 14:626-631, doi: [10.3201/eid1404.071439](https://doi.org/10.3201/eid1404.071439)

* _Cantuti-Castelvetri L_, _Ojha R_ _et&nbsp;al._ (2020) **Neuropilin-1 facilitates SARS-CoV-2 cell entry and infectivity**. Science 370:856-860, doi: [10.1126/science.abd2985](https://doi.org/10.1126/science.abd2985)

* _Yan R_, _Zhang Y_ _et&nbsp;al._ (2020) **Structural basis for the recognition of SARS-CoV-2 by full-length human ACE2**. Science 367:1444-1448, doi: [10.1126/science.abb2762](https://doi.org/10.1126/science.abb2762)

* _Wrapp D_, _Wang N_ _et&nbsp;al._ (2020) **Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation**. Science 367:1260-1263, doi: [10.1126/science.abb2507](https://doi.org/10.1126/science.abb2507)

* _Corman VM_, _Landt O_ _et&nbsp;al._ (2020) **Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR**. Eurosurveillance 25, doi: [10.2807/1560-7917.es.2020.25.3.2000045](https://doi.org/10.2807/1560-7917.es.2020.25.3.2000045)

* _Drosten C_, _Günther S_ _et&nbsp;al._ (2003) **Identification of a Novel Coronavirus in Patients with Severe Acute Respiratory Syndrome**. New England Journal of Medicine 348:1967-1976, doi: [10.1056/nejmoa030747](https://doi.org/10.1056/nejmoa030747)

* _Callaway E_ (2020) **The race for coronavirus vaccines: A graphical guide**. Nature 580:576-577, doi: [10.1038/d41586-020-01221-y](https://doi.org/10.1038/d41586-020-01221-y)

* _Chuong EB_, _Elde NC_ & _Feschotte C_ (2016) **Regulatory evolution of innate immunity through co-option of endogenous retroviruses**. Science 351:1083-1087, doi: [10.1126/science.aad5497](https://doi.org/10.1126/science.aad5497)

* _Herelle F_ (1922) **Der Bakteriophage und seine Bedeutung für die Immunität**. F. Vieweg & Sohn

* _Ball P_ (2020) **The lightning-fast quest for COVID vaccines – and what it means for other diseases**. Nature 589:16-18, doi: [10.1038/d41586-020-03626-1](https://doi.org/10.1038/d41586-020-03626-1)

* _Amit S_, _Regev-Yochay G_ _et&nbsp;al._ (2021) **Early rate reductions of SARS-CoV-2 infection and COVID-19 in BNT162b2 vaccine recipients**. The Lancet 397:875-877, doi: [10.1016/s0140-6736(21)00448-7](https://doi.org/10.1016/s0140-6736(21)00448-7)

* _Aschwanden C_ (2021) **Five reasons why COVID herd immunity is probably impossible**. Nature 591:520-522, doi: [10.1038/d41586-021-00728-2](https://doi.org/10.1038/d41586-021-00728-2)

* _Desmond A_ & _Offit PA_ (2021) **On the Shoulders of Giants – From Jenner's Cowpox to mRNA Covid Vaccines**. N Engl J Med 384:1081-1083, doi: [10.1056/nejmp2034334](https://doi.org/10.1056/nejmp2034334)

* _Mulligan R_ & _Berg P_ (1980) **Expression of a bacterial gene in mammalian cells**. Science 209:1422-1427, doi: [10.1126/science.6251549](https://doi.org/10.1126/science.6251549)

* _Pollard AJ_ & _Bijker EM_ (2020) **A guide to vaccinology: From basic principles to new developments**. Nat Rev Immunol 21:83-100, doi: [10.1038/s41577-020-00479-7](https://doi.org/10.1038/s41577-020-00479-7)

* _Wolff J_, _Malone R_ _et&nbsp;al._ (1990) **Direct gene transfer into mouse muscle in vivo**. Science 247:1465-1468, doi: [10.1126/science.1690918](https://doi.org/10.1126/science.1690918)

* _Jazayeri SD_ & _Poh CL_ (2019) **Recent advances in delivery of veterinary DNA vaccines against avian pathogens**. Veterinary Research 50, doi: [10.1186/s13567-019-0698-z](https://doi.org/10.1186/s13567-019-0698-z)

* _Arashkia A_, _Jalilvand S_ _et&nbsp;al._ (2020) **Severe acute respiratory syndrome-coronavirus-2 spike (S) protein based vaccine candidates: State of the art and future prospects**. Rev Med Virol 31, doi: [10.1002/rmv.2183](https://doi.org/10.1002/rmv.2183)

* _Renz A_, _Widerspick L_ & _Dräger A_ (2020) **FBA reveals guanylate kinase as a potential target for antiviral therapies against SARS-CoV-2**. Bioinformatics 36:i813-i821, doi: [10.1093/bioinformatics/btaa813](https://doi.org/10.1093/bioinformatics/btaa813)

* _Initiative CHG_ (2021) **Mapping the human genetic architecture of COVID-19**. Nature doi: [10.1038/s41586-021-03767-x](https://doi.org/10.1038/s41586-021-03767-x)

* _Group TSCG_ (2020) **Genomewide Association Study of Severe Covid-19 with Respiratory Failure**. N Engl J Med 383:1522-1534, doi: [10.1056/nejmoa2020283](https://doi.org/10.1056/nejmoa2020283)

* _Zeberg H_ & _Pääbo S_ (2020) **The major genetic risk factor for severe COVID-19 is inherited from Neanderthals**. Nature 587:610-612, doi: [10.1038/s41586-020-2818-3](https://doi.org/10.1038/s41586-020-2818-3)

* &#8212; (2021) **A genomic region associated with protection against severe COVID-19 is inherited from Neandertals**. Proc Natl Acad Sci USA 118:e2026309118, doi: [10.1073/pnas.2026309118](https://doi.org/10.1073/pnas.2026309118)

* _Pairo-Castineira E_,  _et&nbsp;al._ (2020) **Genetic mechanisms of critical illness in COVID-19**. Nature 591:92-98, doi: [10.1038/s41586-020-03065-y](https://doi.org/10.1038/s41586-020-03065-y)

* _Vennema H_ (1999) **Genetic drift and genetic shift during feline coronavirus evolution**. Veterinary Microbiology 69:139-141, doi: [10.1016/s0378-1135(99)00102-9](https://doi.org/10.1016/s0378-1135(99)00102-9)

* _Zhang Y_, _Li J_ _et&nbsp;al._ (2015) **Genotype shift in human coronavirus OC43 and emergence of a novel genotype by natural recombination**. Journal of Infection 70:641-650, doi: [10.1016/j.jinf.2014.12.005](https://doi.org/10.1016/j.jinf.2014.12.005)

* _Elbe S_ & _Buckland-Merrett G_ (2017) **Data, disease and diplomacy: GISAIDs innovative contribution to global health**. Global Challenges 1:33-46, doi: [10.1002/gch2.1018](https://doi.org/10.1002/gch2.1018)

* _Konings F_, _Perkins MD_ _et&nbsp;al._ (2021) **SARS-CoV-2 Variants of Interest and Concern naming scheme conducive for global discourse**. Nat Microbiol doi: [10.1038/s41564-021-00932-w](https://doi.org/10.1038/s41564-021-00932-w)

* _Horie M_, _Honda T_ _et&nbsp;al._ (2010) **Endogenous non-retroviral RNA virus elements in mammalian genomes**. Nature 463:84-87, doi: [10.1038/nature08695](https://doi.org/10.1038/nature08695)

* _Mi S_, _Lee X_ _et&nbsp;al._ (2000) **Syncytin is a captive retroviral envelope protein involved in human placental morphogenesis**. Nature 403:785-789, doi: [10.1038/35001608](https://doi.org/10.1038/35001608)

* _Zhang L_, _Richards A_ _et&nbsp;al._ (2021) **Reverse-transcribed SARS-CoV-2 RNA can integrate into the genome of cultured human cells and can be expressed in patient-derived tissues**. Proc Natl Acad Sci USA 118:e2105968118, doi: [10.1073/pnas.2105968118](https://doi.org/10.1073/pnas.2105968118)

* _Kistler KE_ & _Bedford T_ (2021) **Evidence for adaptive evolution in the receptor-binding domain of seasonal coronaviruses OC43 and 229E. eLife** 10, doi: [10.7554/elife.64509](https://doi.org/10.7554/elife.64509)

* _Marshall M_ (2020) **The lasting misery of coronavirus long-haulers**. Nature 585:339-341, doi: [10.1038/d41586-020-02598-6](https://doi.org/10.1038/d41586-020-02598-6)

* _Arnold C_ (2020) **Could COVID delirium bring on dementia?** Nature 588:22-24, doi: [10.1038/d41586-020-03360-8](https://doi.org/10.1038/d41586-020-03360-8)

* _Rhea EM_, _Logsdon AF_ _et&nbsp;al._ (2020) **The S1 protein of SARS-CoV-2 crosses the blood-brain barrier in mice**. Nature Neuroscience doi: [10.1038/s41593-020-00771-8](https://doi.org/10.1038/s41593-020-00771-8)
</details>

<details>
  <summary>Kapitel 16: Zehn innovative Methoden im Bereich der Bio- und Gentechnologie</summary>

* _Editorial_ (2021) **Method of the year 2020: Spatially resolved transcriptomics**. Nature Methods 18:1-1, doi: [10.1038/s41592-020-01042-x](https://doi.org/10.1038/s41592-020-01042-x)

* &#8212; (2020) **Method of the year 2019: Single-cell multimodal omics**. Nature Methods 17:1-1, doi: [10.1038/s41592-019-0703-5](https://doi.org/10.1038/s41592-019-0703-5)

* &#8212; (2018) **Method of the year 2018: Imaging in freely behaving animals**. Nature Methods 16:1-1, doi: [10.1038/s41592-018-0292-8](https://doi.org/10.1038/s41592-018-0292-8)

* &#8212; (2018) **Method of the year 2017: Organoids**. Nature Methods 15:1-1, doi: [10.1038/nmeth.4575](https://doi.org/10.1038/nmeth.4575)

* &#8212; (2017) **Method of the year 2016: Epitranscriptome analysis**. Nature Methods 14:1-1, doi: [10.1038/nmeth.4142](https://doi.org/10.1038/nmeth.4142)

* _Nakane T_, _Kotecha A_ _et&nbsp;al._ (2020) **Single-particle cryo-EM at atomic resolution**. Nature 587:152-156, doi: [10.1038/s41586-020-2829-0](https://doi.org/10.1038/s41586-020-2829-0)

* _Editorial_ (2015) **Method of the year 2015**. Nature Methods 13:1-1, doi: [10.1038/nmeth.3730](https://doi.org/10.1038/nmeth.3730)

* &#8212; (2014) **Method of the year 2014**. Nature Methods 12:1-1, doi: [10.1038/nmeth.3251](https://doi.org/10.1038/nmeth.3251)

* &#8212; (2013) **Method of the year 2013**. Nature Methods 11:1-1, doi: [10.1038/nmeth.2801](https://doi.org/10.1038/nmeth.2801)

* &#8212; (2013) **Method of the year 2012**. Nature Methods 10:1-1, doi: [10.1038/nmeth.2329](https://doi.org/10.1038/nmeth.2329)

* &#8212; (2011) **Method of the year 2011**. Nature Methods 9:1-1, doi: [10.1038/nmeth.1852](https://doi.org/10.1038/nmeth.1852)

* _Sahel JA_, _Boulanger-Scemama E_ _et&nbsp;al._ (2021) **Partial recovery of visual function in a blind patient after optogenetic therapy**. Nature Medicine 27:1223-1229, doi: [10.1038/s41591-021-01351-4](https://doi.org/10.1038/s41591-021-01351-4)

* _Editorial_ (2010) **Method of the year 2010**. Nature Methods 8:1-1, doi: [10.1038/nmeth.f.321](https://doi.org/10.1038/nmeth.f.321)
</details>
